<base href="http://www.themedicalbiochemistrypage.org/enzyme-kinetics.html">
﻿﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Enzymes, Kinetics and Diagnostic Use</title>
<link rel="stylesheet" type="text/css" href="/css/tmbp-pagesstyle.css" />
<style type="text/css">table {
				border-color:#000; border-collapse:collapse;
}
td {
				border-color:#000;
}</style>
<meta name="description" content="The enzyme kinetics page discusses the classification, function, and regulation of the biochemical catalysts." />
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-4746500-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body>

<div id="contents">
<a name="top"></a>
<script> google_analytics_domain_name=".themedicalbiochemistrypage.org"  </script>

<div id="paragraph">
<center>
<div id="table1">
   <a href="/enzyme-kinetics.php#intro">Introduction to Enzymes</a><br />
   <a href="/enzyme-kinetics.php#classification">Enzyme Classifications</a><br />
   <a href="/enzyme-kinetics.php#coenzymes">Role of Coenzymes</a><br />
   <a href="/enzyme-kinetics.php#activities">Enzyme Activity Relative to Substrate Type</a><br />
   <a href="/enzyme-kinetics.php#interactions">Enzyme-Substrate Interactions</a><br />
   <a href="/enzyme-kinetics.php#reactions">Chemical Reactions and Rates</a><br />
   <a href="/enzyme-kinetics.php#order">Chemical Reaction Order</a><br />
   <a href="/enzyme-kinetics.php#catalysts">Enzymes as Biological Catalysts</a><br />
   <a href="/enzyme-kinetics.php#michaelis">Michaelis-Menten Kinetics</a><br />
   <a href="/enzyme-kinetics.php#inhibitors">Inhibition of Enzyme Catalyzed Reactions</a><br />
   <a href="/enzyme-kinetics.php#regulation">Regulation of Enzyme Activity</a><br />
   <a href="/enzyme-kinetics.php#allosteric">Allosteric Enzymes</a><br />
   <a href="/enzyme-kinetics.php#clinical">Enzymes in the Diagnosis of Pathology</a>
 </div>
 </center>
 <br />
<div id="search">
<form action="http://www.google.com" id="cse-search-box" target="_blank">
  <div>
    <input type="hidden" name="cx" value="partner-pub-0170489783089944:7561872190" />
    <input type="hidden" name="ie" value="UTF-8" />
    <input type="text" name="q" size="50" />
    <input type="submit" name="sa" value="Search" />
  </div>
</form>

<script type="text/javascript" src="http://www.google.com/coop/cse/brand?form=cse-search-box&amp;lang=">
</script>
</div>
<br />
<div id="search2">
<!-- start of freefind search box html -->
<table cellpadding=0 cellspacing=0 border=0 >
<tr>
	<td  style="font-family: Arial, Helvetica, sans-serif; font-size: 7.5pt;">
		<center><table width="90%" cellpadding=0 cellspacing=0 border=0  style="font-family: Arial, Helvetica, sans-serif; font-size: 7.5pt;" >
		<tr>
			<td style="font-family: Arial, Helvetica, sans-serif; font-size: 7.5pt;" align=left ><a href="http://search.freefind.com/siteindex.html?si=36599028">index</a></td>
			<td style="font-family: Arial, Helvetica, sans-serif; font-size: 7.5pt;" align=center><a href="http://search.freefind.com/find.html?si=36599028&amp;m=0&amp;p=0">sitemap</a></td>
			<td style="font-family: Arial, Helvetica, sans-serif; font-size: 7.5pt;" align=right><a href="http://search.freefind.com/find.html?si=36599028&amp;pid=a">advanced</a></td>
		</tr>
		</table></center>
		<form style="margin:0px; margin-top:4px;" action="http://search.freefind.com/find.html" method="get" accept-charset="utf-8" target="_blank">
		<input type="hidden" name="si" value="36599028">
		<input type="hidden" name="pid" value="r">
		<input type="hidden" name="n" value="0">
		<input type="hidden" name="_charset_" value="">
		<input type="hidden" name="bcd" value="&#247;">
		<input type="text" name="query" size="15"> 
		<input type="submit" value="search">
		</form>
	</td>
</tr>
<tr>
	<td style="text-align:center; font-family: Arial, Helvetica, sans-serif;	font-size: 7.5pt; padding-top:4px;">
		<a style="text-decoration:none; color:gray;" href="http://www.freefind.com" >site search</a><a style="text-decoration:none; color:gray;" href="http://www.freefind.com" > by
		<span style="color: #606060;">freefind</span></a>
	</td>
</tr>
</table>
<!-- end of freefind search box html -->
</div>

<h3><a href="/index.php">Return to The Medical Biochemistry Page</a> </h3>
<center><span style="font-size: 10pt">&#169; 1996&ndash;2013 themedicalbiochemistrypage.org, LLC &#124; info @ 
themedicalbiochemistrypage.org</span>
</center>
<hr />

<h1><a name="intro"></a>Introduction to Enzymes</h1>
<h3>
<script type="text/javascript">
<!--
google_ad_client = "ca-pub-0170489783089944";
/* TMBP pages links 6-7-12 */
google_ad_slot = "9660312975";
google_ad_width = 728;
google_ad_height = 15;
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</h3>

<div style="float:right" class="rectangleRight">
<script type="text/javascript"><!--
google_ad_client = "pub-0170489783089944";
/* 160x600, created 8/27/10 */
google_ad_slot = "1175766666";
google_ad_width = 160;
google_ad_height = 600;
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</div>
<p>Enzymes are biological catalysts responsible for supporting almost all of the
chemical reactions that maintain animal homeostasis. Because of their role in
maintaining life processes, the assay and pharmacological regulation of enzymes
have become key elements in clinical diagnosis and therapeutics. The
macromolecular components of almost all enzymes are composed of protein, except
for a class of RNA catalysts known as <strong>ribozymes</strong>. The 
term ribozyme is derived from <strong>ribo</strong>nucleic acid en<strong>zyme</strong>. 
Ribozymes are molecules of ribonucleic acid that catalyze reactions on one of their own 
phosphodiester bonds or within other RNAs. In addition, there are known ribozymes that 
catalyze reactions on proteins, the best example being the peptidyltransferase activity 
of the large ribosomal subunit of <a href="/protein-synthesis.php">protein synthesis</a>.</p>
<p>Enzymes are found in all tissues and fluids of the
body. Intracellular enzymes catalyze the reactions of metabolic pathways.
Plasma membrane enzymes regulate catalysis within cells in response to
extracellular signals, and enzymes of the circulatory system are responsible
for regulating the <a href="/blood-coagulation.php">clotting of blood</a>.
Almost every significant life process is dependent on enzyme activity.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="classification"></a>Enzyme Classifications</h1>
<p>Traditionally, enzymes were simply assigned names by
the investigator who discovered the enzyme. As knowledge expanded, systems of
enzyme classification became more comprehensive and complex. Currently enzymes
are grouped into six functional classes by the International Union of
Biochemists (I.U.B.).</p>

<table border="1" cellpadding="10px">
 <tr>
  <td class="tableheads">Number
  </td>
  <td class="tableheads">Classification
  </td>
  <td class="tableheads">Biochemical Properties
  </td>
 </tr>
 <tr>
  <td align="center">
  1
  </td>
  <td align="center">
  Oxidoreductases  </td>
  <td align="left">
  Act on many chemical
  groupings to add or remove hydrogen atoms.  </td>
 </tr>
 <tr>
  <td align="center">
  2  </td>
  <td align="center">
 Transferases  </td>
  <td align="left">
  Transfer functional groups
  between donor and acceptor molecules. Kinases are specialized transferases
  that regulate metabolism by transferring phosphate from ATP to other
  molecules.  </td>
 </tr>
 <tr>
  <td align="center">
  3</td>
  <td align="center">
  Hydrolases  </td>
  <td align="left">
 Add water across a bond,
  hydrolyzing it.  </td>
 </tr>
 <tr>
  <td align="center">
  4  </td>
  <td align="center">
  Lyases  </td>
  <td align="left">
  Add water, ammonia or
  carbon dioxide across double bonds, or remove these elements to produce
  double bonds.  </td>
 </tr>
 <tr>
  <td align="center">
  5</td>
  <td align="center">
  Isomerases  </td>
  <td align="left">
  Carry out many kinds of isomerization:
  L to D isomerizations, mutase reactions (shifts of chemical groups) and
  others.  </td>
 </tr>
 <tr>
  <td align="center">
  6  </td>
  <td align="center">
 Ligases  </td>
  <td align="left">
  Catalyze reactions in
  which two chemical groups are joined (or ligated) with the use of energy from ATP.
  </td>
 </tr>
</table>

<p>These rules give each enzyme a unique number. The
I.U.B. system also specifies a textual name for each enzyme. The enzyme's name
is comprised of the names of the substrate(s), the product(s) and the enzyme's
functional class. Because many enzymes, such as alcohol dehydrogenase, are
widely known in the scientific community by their common names, the change to
I.U.B.-approved nomenclature has been slow. In everyday usage, most enzymes are
still called by their common name.</p>
<h3>
<div style="float:left" class="rectangleLeft">
<script type="text/javascript">
<!--
google_ad_client = "ca-pub-0170489783089944";
/* TMBP pages rectangle 6-7-12 */
google_ad_slot = "8229094811";
google_ad_width = 300;
google_ad_height = 250;
//-->
</script>

<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">

</script>
</div>
<div style="float:right" class="rectangleRight">
<script type="text/javascript">
<!--
google_ad_client = "ca-pub-0170489783089944";
/* TMBP pages rectangle 6-7-12 */
google_ad_slot = "8229094811";
google_ad_width = 300;
google_ad_height = 250;
//-->
</script>

<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">

</script>
</div>
</h3>
<p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>


<p>Enzymes are also classified on the basis of their
composition. Enzymes composed wholly of protein are known as simple
enzymes in contrast to complex enzymes, which are composed of
protein plus a relatively small organic molecule. Complex enzymes are also
known as <strong>holoenzymes</strong>. In this terminology the protein component is known
as the <strong>apoenzyme</strong>, while the non-protein component is known as the <strong>coenzyme</strong>
or <strong>prosthetic group</strong> where prosthetic group describes a complex in which
the small organic molecule is bound to the apoenzyme by covalent bonds; when
the binding between the apoenzyme and non-protein components is non-covalent,
the small organic molecule is called a <strong>coenzyme</strong>. Many prosthetic groups
and coenzymes are water-soluble derivatives of <a href="/vitamins.php">vitamins</a>. 
It should be noted that the main clinical symptoms of dietary vitamin insufficiency generally arise from the
malfunction of enzymes, which lack sufficient cofactors derived from vitamins
to maintain homeostasis.</p>

<p>The non-protein component of an
enzyme may be as simple as a metal ion or as complex as a small non-protein
organic molecule. Enzymes that require a metal in their composition are known
as <strong>metalloenzymes</strong> if they bind
and retain their metal atom(s) under all conditions with very high affinity.
Those which have a lower affinity for metal ion, but still require the metal
ion for activity, are known as metal-activated enzymes.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="coenzymes"></a>Role of Coenzymes</h1>

<p>The functional role of coenzymes is to act as transporters
of chemical groups from one reactant to another. The chemical groups carried
can be as simple as the hydride ion (H<sup>+</sup> + 2<em>e</em><sup>&ndash;</sup>)
carried by NAD or the mole of hydrogen carried by FAD; or they can be even more
complex than the amine (&ndash;NH<sub>2</sub>) carried by pyridoxal phosphate.</p>

<p>Since coenzymes are chemically changed as a
consequence of enzyme action, it is often useful to consider coenzymes to be a
special class of substrates, or second substrates, which are common to many different holoenzymes. In
all cases, the coenzymes donate the carried chemical grouping to an acceptor
molecule and are thus regenerated to their original form. This regeneration of
coenzyme and holoenzyme fulfills the definition of an enzyme as a chemical
catalyst, since (unlike the usual substrates, which are used up during the
course of a reaction) coenzymes are generally regenerated.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="activities"></a>Enzyme Relative to Substrate Type</h1>

<p>Although enzymes are highly specific for the kind of
reaction they catalyze, the same is not always true of substrates they attack.
For example, while succinate dehydrogenase (SDH) always catalyzes an
oxidation-reduction reaction and its substrate is invariably succinic acid,
alcohol dehydrogenase (ADH) always catalyzes oxidation-reduction reactions but
attacks a number of different alcohols, ranging from methanol to butanol.
Generally, enzymes having broad substrate specificity are most active against
one particular substrate. In the case of ADH, ethanol is the preferred
substrate.</p>

<p>Enzymes also are generally specific for a particular
steric configuration (optical isomer) of a substrate. Enzymes that attack D
sugars will not attack the corresponding L isomer. Enzymes that act on L amino
acids will not employ the corresponding D optical isomer as a substrate. The
enzymes known as racemases provide a striking exception to these generalities;
in fact, the role of racemases is to convert D isomers to L isomers and vice
versa. Thus, racemases attack both D and L forms of their substrate.</p>

<p>As enzymes have a more or less broad range of
substrate specificity, it follows that a given substrate may be acted on by a
number of different enzymes, each of which uses the same substrate(s) and
produces the same product(s). The individual members of a set of enzymes
sharing such characteristics are known as <strong>isozymes</strong>.
These are the products of genes that vary only slightly; often, various
isozymes of a group are expressed in different tissues of the body. The best
studied set of isozymes is the lactate dehydrogenase (LDH) system. LDH is a
tetrameric enzyme composed of all possible arrangements of two different
protein subunits; the subunits are known as H (for heart) and M (for skeletal
muscle). These subunits combine in various combinations leading to 5 distinct
isozymes. The all H isozyme is characteristic of that from heart tissue, and
the all M isozyme is typically found in skeletal muscle and liver. These
isozymes all catalyze the same chemical reaction, but they exhibit differing
degrees of efficiency. The detection of specific LDH isozymes in the blood is 
highly diagnostic of tissue damage such as occurs during cardiac infarct 
(<a href="/enzyme-kinetics.php#clinical">see below</a>).</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="interactions"></a>Enzyme-Substrate Interactions</h1>
<h3>
<script type="text/javascript">
<!--
google_ad_client = "ca-pub-0170489783089944";
/* TMBP pages links 6-7-12 */
google_ad_slot = "9660312975";
google_ad_width = 728;
google_ad_height = 15;
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</h3>
<p>The favored model of enzyme-substrate interaction is
known as the <strong>induced-fit model</strong>. This model
proposes that the initial interaction between enzyme and substrate is
relatively weak, but that these weak interactions rapidly induce conformational
changes in the enzyme that strengthen binding and bring catalytic sites close
to substrate bonds to be altered. After binding takes place, one or more
mechanisms of catalysis generate transition-state complexes and reaction
products. The possible mechanisms of catalysis are four in number:</p>

<p class="para2"><strong>1. Catalysis by Bond Strain:</strong> In this
form of catalysis, the induced structural rearrangements that take place with
the binding of substrate and enzyme ultimately produce strained substrate
bonds, which more easily attain the transition state. The new conformation
often forces substrate atoms and bulky catalytic groups, such as aspartate and
glutamate, into conformations that strain existing substrate bonds.</p>

<p class="para2"><strong>2. Catalysis by Proximity and Orientation:</strong> 
Enzyme-substrate interactions orient reactive groups and bring them
into proximity with one another. In addition to inducing strain, groups such as
aspartate are frequently chemically reactive as well, and their proximity and
orientation toward the substrate thus favors their participation in catalysis.</p>

<p class="para2"><strong>3. Catalysis Involving Proton Donors (Acids) and 
Acceptors (Bases):</strong> Other mechanisms also contribute significantly to
the completion of catalytic events initiated by a strain mechanism, for
example, the use of glutamate as a general acid catalyst (proton donor).</p>

<p class="para2"><strong>4. Covalent Catalysis:</strong> In catalysis
that takes place by covalent mechanisms, the substrate is oriented to active
sites on the enzymes in such a way that a covalent intermediate forms between
the enzyme or coenzyme and the substrate. One of the best-known examples of
this mechanism is that involving proteolysis by serine proteases, which include both digestive enzymes (trypsin,
chymotrypsin, and elastase) and several enzymes of the 
<a href="/blood-coagulation.php">blood clotting cascade</a>. These proteases
contain an active site serine whose R group hydroxyl forms a covalent bond with
a carbonyl carbon of a peptide bond, thereby causing hydrolysis of the peptide
bond.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="reactions"></a>Chemical Reactions and Rates</h1>

<p>According to the conventions of biochemistry, the
rate of a chemical reaction is described by the number of molecules of
reactant(s) that are converted into product(s) in a specified time period. Reaction
rate is always dependent on the concentration of the chemicals involved in the
process and on rate constants that are characteristic of the reaction. For
example, the reaction in which A is converted to B is written as follows:</p>

<h3>A &mdash;&mdash;&gt; B</h3>

<p>The rate of this reaction is expressed algebraically
as either a decrease in the concentration of reactant A:</p>

<h3>&ndash;[A] = <em>k</em>[B]</h3>

<p>or an increase in the concentration of product B:</p>

<h3>[B] = <em>k</em>[A]</h3>

<p>In the second equation (of the 3 above) the negative
sign signifies a decrease in concentration of A as the reaction progresses,
brackets define concentration in molarity and the <em>k</em> is known as a rate
constant. Rate constants are simply proportionality constants that provide a
quantitative connection between chemical concentrations and reaction rates.
Each chemical reaction has characteristic values for its rate constants; these
in turn directly relate to the <strong>equilibrium constant</strong> for that reaction. Thus, reaction can be rewritten
as an equilibrium expression in order to show the relationship between reaction
rates, rate constants and the equilibrium constant for this simple case. The
rate constant for the forward reaction is defined as k<sub>+1</sub> and the
reverse as k<sub>&ndash;1</sub>.</p>

<p>At equilibrium the rate (v) of the forward reaction (A
&mdash;&mdash;&gt; B) is, by definition, equal to that of the reverse or back
reaction (B &mdash;&mdash;&gt; A), a relationship which is algebraically symbolized as: </p>

<h3>v<sub>forward</sub> = v<sub>reverse</sub></h3>

<p>where, for the forward reaction: </p>

<h3>v<sub>forward</sub> = k<sub>+1</sub>[A]</h3>

<p>and for the reverse reaction:</p>

<h3>v<sub>reverse</sub> = k<sub>&ndash;1</sub>[B]</h3>

<p>In the above equations, k<sub>+1</sub> and k<sub>&ndash;1</sub>
represent rate constants for the forward and reverse reactions, respectively.
The negative subscript refers only to a reverse reaction, not to an actual
negative value for the constant. To put the relationships of the two equations
into words, we state that the rate of the forward reaction [v<sub>forward</sub>]
is equal to the product of the forward rate constant k<sub>+1</sub> and the
molar concentration of A. The rate of the reverse reaction is equal to the
product of the reverse rate constant k<sub>&ndash;1</sub> and the molar concentration
of B.</p>

<p>At equilibrium, the rate of the forward reaction is
equal to the rate of the reverse reaction leading to the <strong>equilibrium
constant</strong> of the reaction and is
expressed by:</p>

<h3><img src="/images/kinetics-keq.jpg" alt="Equation of the equilibrium constant, Keq" title="Equation of the equilibrium constant, Keq" /></h3>

<p>This
equation demonstrates that the equilibrium constant for a chemical reaction is
not only equal to the equilibrium ratio of product and reactant concentrations,
but is also equal to the ratio of the characteristic rate constants of the
reaction.</p>

<h4>Examples of Coupled Reactions in Biology</h4>
<p>Let's look at the hypothetical reaction: A &mdash;&mdash;&gt; B. If this is a 
thermodynamically unfavorable reaction the &#916;G<sup>o'</sup> value will be positive. Let's assume it is +4.0 kcal/mol. 
In order to drive this reaction in the direction written it can be coupled to the hydrolysis of ATP. The free energy 
of ATP hydrolysis to ADP is shown:</p>
<p class="para3">ATP + H<sub>2</sub>O &mdash;&mdash;&gt; ADP + P<sub>i</sub>: &#916;G<sup>o'</sup> = &ndash;7.3 kcal/mol</p>
<p>Coupling the two reactions together gives the equation:</p>
<p class="para3">A + ATP + H<sub>2</sub>O &mdash;&mdash;&gt; B + ADP + P<sub>i</sub> + H<sup>+</sup></p>
<p>The &#916;G<sup>o'</sup> for this coupled reaction is the sum of the &#916;G<sup>o'</sup> values of the two separate 
reactions, i.e. (-7.3kcal/mol) + (+4.0kcal/mol) = -3.3kcal/mol. This indicates that coupling ATP hydrolysis 
provides the energy necessary to make the conversion of A to B thermodynamically favorable.</p>
<p>Another useful example is to examine one of the reactions of 
<a href="/glycolysis.php">glycolysis</a>. In this case we will look at 
the oxidation of phosphoenolpyruvate to pyruvate catalyzed by the enzyme 
pyruvate kinase (PK).</p>
<p class="para3">phosphoenolpyruvate &mdash;&mdash;&gt; pyruvate: &#916;G<sup>o'</sup> = &ndash;14.7 kcal/mol</p>
<p>This reaction releases sufficient energy to drive 
the synthesis of ATP from ADP which would normally be thermodynamically 
unfavorable with a &#916;G<sup>o'</sup> of +7.3kcal/mol. Note that this value is the reciprocal 
of the hydrolysis of ATP. This points out another fact that the &#916;G<sup>o'</sup> for a reaction 
in one direction is equal but mathematically opposite for the reciprocal direction. Coupling the two 
reactions together yields:</p>
<p class="para3">phosphoenolpyruvate + ADP + H<sup>+</sup> &mdash;&mdash;&gt; pyruvate + ATP: &#916;G<sup>o'</sup> = &ndash;7.4 kcal/mol</p>

<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="order"></a>Chemical Reaction Order</h1>

<p>Reaction order refers to the number of molecules
involved in forming a reaction complex that is competent to proceed to
product(s). Empirically, order is easily determined by summing the exponents of
each concentration term in the rate equation for a reaction. A reaction
characterized by the conversion of one molecule of A to one molecule of B with
no influence from any other reactant or solvent is a <strong>first-order</strong> reaction. The exponent on the substrate
concentration in the rate equation for this type of reaction is 1. A reaction
with two substrates forming two products would a <strong>second-order</strong> reaction. 
However, the reactants in second&ndash; and
higher&ndash; order reactions need not be different chemical species. An example of a
second order reaction is the formation of ATP through the condensation of ADP
with orthophosphate:</p>

<h3>ADP + H<sub>2</sub>PO<sub>4</sub><sup>&ndash;</sup>
&lt;&mdash;&mdash;&gt; ATP + H<sub>2</sub>O</h3>

<p>For this reaction the forward reaction rate would be
written as: </p>

<h3>v<sub>forward</sub> = k<sub>1</sub>[ADP][H<sub>2</sub>PO<sub>4</sub><sup>&ndash;</sup>]</h3>

<br />
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="catalysts"></a>Enzymes as Biological Catalysts</h1>

<p>In cells and organisms most reactions are catalyzed
by enzymes, which are regenerated during the course of a reaction. These biological
catalysts are physiologically important
because they speed up the rates of reactions that would otherwise be too slow
to support life. Enzymes increase reaction rates, sometimes by as much as one million-fold, 
but more typically by about one thousand fold. Catalysts speed up
the forward and reverse reactions proportionately so that, although the
magnitude of the rate constants of the forward and reverse reactions is are
increased, the ratio of the rate constants remains the same in the presence or
absence of enzyme. Since the equilibrium constant is equal to a ratio of rate
constants, it is apparent that enzymes and other catalysts have no effect on
the equilibrium constant of the reactions they catalyze.</p>

<p>Enzymes increase reaction rates by decreasing the
amount of energy required to form a complex of reactants that is competent to
produce reaction products. This complex is known as the activated state or transition
state complex for the reaction. Enzymes
and other catalysts accelerate reactions by lowering the energy of the
transition state. The free energy required to form an activated complex is much
lower in the catalyzed reaction. The amount of energy required to achieve the
transition state is lowered; consequently, at any instant a greater proportion
of the molecules in the population can achieve the transition state. The result
is that the reaction rate is increased.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="michaelis"></a>Michaelis-Menten Kinetics</h1>

<p>In typical enzyme-catalyzed reactions, reactant and
product concentrations are usually hundreds or thousands of times greater than
the enzyme concentration. Consequently, each enzyme molecule catalyzes the
conversion to product of many reactant molecules. In biochemical reactions,
reactants are commonly known as substrates. The catalytic event that converts
substrate to product involves the formation of a transition state, and it
occurs most easily at a specific binding site on the enzyme. This site, called
the catalytic site of the enzyme, has
been evolutionarily structured to provide specific, high-affinity binding of
substrate(s) and to provide an environment that favors the catalytic events.
The complex that forms, when substrate(s) and enzyme combine, is called the enzyme
substrate (ES) complex. Reaction
products arise when the ES complex breaks down releasing free enzyme.</p>

<p>Between the binding of substrate to enzyme, and the
reappearance of free enzyme and product, a series of complex events must take
place. At a minimum an ES complex must be formed; this complex must pass to the
transition state (ES*); and the transition state complex must advance to an
enzyme product complex (EP). The latter is finally competent to dissociate to
product and free enzyme. The series of events can be shown thus:</p>

<h3>E + S &lt;&mdash;&mdash;&gt; ES &lt;&mdash;&mdash;&gt; ES*
&lt;&mdash;&mdash;&gt; EP &lt;&mdash;&mdash;&gt; E + P</h3>

<p>The kinetics of simple reactions like that above were
first characterized by biochemists Michaelis and Menten. The concepts
underlying their analysis of enzyme kinetics continue to provide the
cornerstone for understanding metabolism today, and for the development and
clinical use of drugs aimed at selectively altering rate constants and
interfering with the progress of disease states. The Michaelis-Menten 
equation is a quantitative description of the relationship
among the rate of an enzyme- catalyzed reaction [v<sub>1</sub>], the
concentration of substrate [S] and two constants, V<sub>max</sub> and K<sub>m</sub>
(which are set by the particular equation). The symbols used in the
Michaelis-Menten equation refer to the reaction rate [v<sub>1</sub>], maximum
reaction rate (V<sub>max</sub>), substrate concentration [S] and the
Michaelis-Menten constant (K<sub>m</sub>).</p>

<h3><img src="/images/michaelismentenvelocityequation.jpg" alt="Michelis-Menten equation relative to initial velocity" title="Michelis-Menten equation relative to initial velocity" />

</h3>

<p>The Michaelis-Menten equation can be used to
demonstrate that at the substrate concentration that produces exactly half of
the maximum reaction rate, i.e. &frac12;&nbsp;V<sub>max</sub>, the substrate
concentration is numerically equal to K<sub>m</sub>. This fact provides a
simple yet powerful bioanalytical tool that has been used to characterize both
normal and altered enzymes, such as those that produce the symptoms of genetic
diseases. Rearranging the Michaelis-Menten equation leads to: </p>

<h3><img src="/images/kmequation.jpg" alt="Derivation of Michelis-Menten constant, Km" title="Derivation of Michelis-Menten constant, Km" /></h3>

<p>From this equation it should be apparent that when
the substrate concentration is half that required to support the maximum rate
of reaction, the observed rate (v<sub>1</sub>) will be equal to V<sub>max</sub>
divided by 2; in other words, v<sub>1</sub> = [V<sub>max</sub>/2]. At this
substrate concentration V<sub>max</sub>/v<sub>1</sub> will be exactly equal to
2, with the result that:</p>

<h3>[S](1) = K<sub>m</sub></h3>

<p>The latter is an algebraic statement of the fact
that, for enzymes of the Michaelis-Menten type, when the observed reaction rate
is half of the maximum possible reaction rate, the substrate concentration is
numerically equal to the Michaelis-Menten constant. In this derivation, the
units of K<sub>m</sub> are those used to specify the concentration of S,
usually Molarity.</p>

<p>The Michaelis-Menten equation has the same form as the equation for a rectangular
hyperbola; graphical analysis of reaction rate (v) versus substrate
concentration [S] produces a hyperbolic rate plot. </p>

<p class="para1b"><img src="/images/enzyme-saturation-curve.jpg" alt="Enzyme-substrate saturation curve" title="Enzyme-substrate saturation curve" /></p>
  <h3 class="h-ridge">Plot of substrate concentration versus reaction
  velocity</h3>

<p>The key features of the plot are marked by points A,
B and C. At high substrate concentrations the rate represented by point C the
rate of the reaction is almost equal to V<sub>max</sub>, and the difference in rate
at nearby concentrations of substrate is almost negligible. If the
Michaelis-Menten plot is extrapolated to infinitely high substrate
concentrations, the extrapolated rate is equal to V<sub>max</sub>. When the
reaction rate becomes independent of substrate concentration, or nearly so, the
rate is said to be zero order. (Note that the reaction is zero order only with
respect to this substrate. If the reaction has two substrates, it may or may
not be zero order with respect to the second substrate). The very small differences
in reaction velocity at substrate concentrations around point C (near V<sub>max</sub>)
reflect the fact that at these concentrations almost all of the enzyme
molecules are bound to substrate and the rate is virtually independent of
substrate, hence zero order. At lower substrate concentrations, such as at
points A and B, the lower reaction velocities indicate that at any moment only
a portion of the enzyme molecules are bound to the substrate. In fact, at the
substrate concentration denoted by point B, exactly half the enzyme molecules
are in an ES complex at any instant and the rate is exactly one half of V<sub>max</sub>.
At substrate concentrations near point A the rate appears to be directly
proportional to substrate concentration, and the reaction rate is said to be
first order.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="inhibitors"></a>Inhibition of Enzyme Catalyzed Reactions</h1>
<h3>
<script type="text/javascript">
<!--
google_ad_client = "ca-pub-0170489783089944";
/* TMBP pages links 6-7-12 */
google_ad_slot = "9660312975";
google_ad_width = 728;
google_ad_height = 15;
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</h3>
<p>To avoid dealing with curvilinear plots of enzyme
catalyzed reactions, biochemists Lineweaver and Burk introduced an analysis of
enzyme kinetics based on the following rearrangement of the Michaelis-Menten
equation:</p>

<h3><img src="/images/kinetics-lineweaver.jpg" alt="Lineweaver-Burk equation" title="Lineweaver-Burk equation" /></h3>

<p>Plots
of 1/v versus 1/[S] yield straight lines having a slope of K<sub>m</sub>/V<sub>max</sub>
and an intercept on the ordinate at 1/V<sub>max</sub>. </p>

<h3><img src="/images/lineweaver-burk-plot.jpg" alt="Lineweaver-Burk plot" title="Lineweaver-Burk plot" /></h3>
  <h3>A Lineweaver-Burk Plot</h3>

<p>An alternative linear transformation of the Michaelis-Menten equation is the
Eadie-Hofstee transformation:</p>

<h3><img src="/images/kinetics-eadie.jpg" alt="Eadie-Hofstee equation" title="Eadie-Hofstee equation" /></h3>

<p>and when v/[S] is
plotted on the y-axis versus v on the x-axis, the result is a linear plot with
a slope of &ndash;1/K<sub>m</sub> and the value V<sub>max</sub>/K<sub>m</sub> as the
intercept on the y-axis and V<sub>max</sub> as the intercept on the x-axis.</p>

<p>Both the Lineweaver-Burk and Eadie-Hofstee transformation
of the Michaelis-Menton equation are useful in the analysis of enzyme
inhibition. Since most clinical drug therapy is based on inhibiting the
activity of enzymes, analysis of enzyme reactions using the tools described
above has been fundamental to the modern design of pharmaceuticals. Well- known
examples of such therapy include the use of 
<a href="/dna.php#chemo">methotrexate in cancer chemotherapy</a>
to semi-selectively inhibit DNA synthesis of malignant cells, the use of
<a href="/aspirin.php">aspirin</a> 
to inhibit the synthesis of prostaglandins which are at least partly
responsible for the aches and pains of arthritis, and the use of sulfa drugs to
inhibit the folic acid synthesis that is essential for the metabolism and
growth of disease-causing bacteria. In addition, many poisons, such as cyanide,
carbon monoxide and polychlorinated biphenols (PCBs) produce their 
life-threatening effects by means of enzyme inhibition.</p>

<p>Enzyme inhibitors fall into two broad classes: those
causing irreversible inactivation of enzymes and those whose inhibitory effects
can be reversed. Inhibitors of the first class usually cause an inactivating,
covalent modification of enzyme structure. Cyanide is a classic example of an
irreversible enzyme inhibitor: by covalently binding mitochondrial cytochrome
oxidase, it inhibits all the reactions associated with electron transport. The
kinetic effect of irreversible inhibitors is to decrease the concentration of
active enzyme, thus decreasing the maximum possible concentration of ES
complex. Since the limiting enzyme reaction rate is often k<sub>2</sub>[ES], it
is clear that under these circumstances the reduction of enzyme concentration
will lead to decreased reaction rates. Note that when enzymes in cells are only
partially inhibited by irreversible inhibitors, the remaining unmodified enzyme
molecules are not distinguishable from those in untreated cells; in particular,
they have the same turnover number and the same K<sub>m</sub>. Turnover
number, related to V<sub>max</sub>, is
defined as the maximum number of moles of substrate that can be converted to
product per mole of catalytic site per second. Irreversible inhibitors are
usually considered to be poisons and are generally unsuitable for therapeutic
purposes.</p>

<p>Reversible inhibitors can be divided into two main categories; 
<strong>competitive inhibitors</strong> and 
<strong>noncompetitive inhibitors</strong>, with a third category, 
<strong>uncompetitive inhibitors</strong>, rarely encountered.</p>


<table border="1" cellpadding="10px">
 <tr>
  <td class="tableheads">Inhibitor Type
  </td>
  <td class="tableheads">Binding Site on Enzyme
  </td>
  <td class="tableheads">Kinetic effect
  </td>
 </tr>
 <tr>
  <td align="center">
  Competitive Inhibitor  </td>
  <td>
  Specifically at the catalytic
  site, where it competes with substrate for binding in a dynamic equilibrium-
  like process. Inhibition is reversible by substrate.  </td>
  <td>
 V<sub>max</sub> is
  unchanged; K<sub>m</sub>, as defined by [S] required for &frac12; maximal
  activity, is increased.  </td>
 </tr>
 <tr>
  <td align="center">
  Noncompetitive Inhibitor  </td>
  <td>
 Binds E or ES complex
  other than at the catalytic site. Substrate binding unaltered, but ESI
  complex cannot form products. Inhibition cannot be reversed by substrate.  </td>
  <td>
  K<sub>m</sub> appears
  unaltered; V<sub>max</sub> is decreased proportionately to inhibitor
  concentration.  </td>
 </tr>
 <tr>
  <td align="center">
  Uncompetitive Inhibitor  </td>
  <td>
  Binds only to ES complexes
  at locations other than the catalytic site. Substrate binding modifies enzyme
  structure, making inhibitor- binding site available. Inhibition cannot be
  reversed by substrate.</td>
  <td>
 Apparent V<sub>max</sub>
  decreased; K<sub>m</sub>, as defined by [S] required for &frac12; maximal
  activity, is decreased.   </td>
 </tr>
</table>

<p>The hallmark of all the reversible inhibitors is that
when the inhibitor concentration drops, enzyme activity is regenerated. Usually
these inhibitors bind to enzymes by non-covalent forces and the inhibitor
maintains a reversible equilibrium with the enzyme. The equilibrium constant
for the dissociation of enzyme inhibitor complexes is known as K<sub>i</sub>: </p>

<h3><img src="/images/kinetics-ki.jpg" alt="Equation for calculating dissociation constant of enzyme-inhibitor complexes" 
title="Equation for calculating dissociation constant of enzyme-inhibitor complexes" /></h3>

<p>The importance of K<sub>I</sub> is that in all enzyme
reactions where substrate, inhibitor and enzyme interact, the normal K<sub>m</sub>
and or V<sub>max</sub> for substrate enzyme interaction appear to be altered.
These changes are a consequence of the influence of K<sub>i</sub> on the
overall rate equation for the reaction. The effects of K<sub>i</sub> are best
observed in Lineweaver-Burk plots. </p>

<p>Probably
the best known reversible inhibitors are competitive inhibitors, which always
bind at the catalytic or active site of the enzyme. Most drugs that alter enzyme
activity are of this type. Competitive inhibitors are especially attractive as
clinical modulators of enzyme activity because they offer two routes for the
reversal of enzyme inhibition, while other reversible inhibitors offer only
one. First, as with all kinds of reversible inhibitors, a decreasing
concentration of the inhibitor reverses the equilibrium regenerating active
free enzyme. Second, since substrate and competitive inhibitors both bind at
the same site they compete with one another for binding.</p>

<p>Raising the concentration of substrate (S), while
holding the concentration of inhibitor constant, provides the second route for
reversal of competitive inhibition. The greater the proportion of substrate,
the greater the proportion of enzyme present in competent ES complexes.</p>

<p>As noted earlier, high concentrations of substrate
can displace virtually all competitive inhibitor bound to active sites. Thus,
it is apparent that V<sub>max</sub> should be unchanged by competitive
inhibitors. This characteristic of competitive inhibitors is reflected in the
identical vertical-axis intercepts of Lineweaver-Burk plots, with and without
inhibitor.</p>

<p class="para1b"><img src="/images/enzyme-inhibition.jpg" alt="Lineweaver-Burk plots of inhibited enzymes" title="Lineweaver-Burk plots of inhibited enzymes" /></p>
  <h3 class="h-ridge">Lineweaver-Burk Plots of Inhibited Enzymes</h3>

<p>Since attaining V<sub>max</sub> requires appreciably
higher substrate concentrations in the presence of competitive inhibitor, K<sub>m</sub>
(the substrate concentration at half maximal velocity) is also higher, as
demonstrated by the differing negative intercepts on the horizontal axis in
panel B.</p>

<p>Analogously, panel C illustrates that noncompetitive
inhibitors appear to have no effect on the intercept at the x-axis implying
that noncompetitive inhibitors have no effect on the K<sub>m</sub> of the
enzymes they inhibit. Since noncompetitive inhibitors do not interfere in the
equilibration of enzyme, substrate and ES complexes, the K<sub>m</sub>'s of
Michaelis-Menten type enzymes are not expected to be affected by noncompetitive
inhibitors, as demonstrated by x-axis intercepts in panel C. However, because
complexes that contain inhibitor (ESI) are incapable of progressing to reaction
products, the effect of a noncompetitive inhibitor is to reduce the
concentration of ES complexes that can advance to product. Since V<sub>max</sub>
= k<sub>2</sub>[E<sub>total</sub>], and the concentration of competent E<sub>total</sub>
is diminished by the amount of ESI formed, noncompetitive inhibitors are
expected to decrease V<sub>max</sub>, as illustrated by the y-axis intercepts
in panel C.</p>

<p>A corresponding analysis of uncompetitive inhibition
leads to the expectation that these inhibitors should change the apparent
values of K<sub>m</sub> as well as V<sub>max</sub>. Changing both constants
leads to double reciprocal plots, in which intercepts on the x and y axes are
proportionately changed; this leads to the production of parallel lines in
inhibited and uninhibited reactions.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="regulation"></a>Regulation of Enzyme Activity</h1>

<p>While it is clear that enzymes are responsible for
the catalysis of almost all biochemical reactions, it is important to also
recognize that rarely, if ever, do enzymatic reactions proceed in isolation.
The most common scenario is that enzymes catalyze individual steps of
multi-step metabolic pathways, as is the case with <a href="/glycolysis.php">glycolysis</a>,
<a href="/gluconeogenesis.php">gluconeogenesis</a> or the 
<a href="/lipid-synthesis.php">synthesis of fatty acids</a>. As a consequence of
these lock- step sequences of reactions, any given enzyme is dependent on the
activity of preceding reaction steps for its substrate.</p>

<p>In
humans, substrate concentration is dependent on food supply and is not usually
a physiologically important mechanism for the routine regulation of enzyme
activity. Enzyme concentration, by contrast, is continually modulated in
response to physiological needs. Three principal mechanisms are known to
regulate the concentration of active enzyme in tissues:</p>

<p class="para2"><strong>1.</strong> Regulation of gene expression controls the quantity
and rate of enzyme synthesis.</p>

<p class="para2"><strong>2.</strong> Proteolytic enzyme activity determines the rate of
enzyme degradation.</p>

<p class="para2"><strong>3.</strong> Covalent modification of preexisting pools of
inactive proenzymes produces active enzymes.</p>

<p>Enzyme synthesis and proteolytic degradation are
comparatively slow mechanisms for regulating enzyme concentration, with
response times of hours, days or even weeks. Proenzyme activation is a more
rapid method of increasing enzyme activity but, as a regulatory mechanism, it
has the disadvantage of not being a reversible process. Proenzymes are
generally synthesized in abundance, stored in secretory granules and covalently
activated upon release from their storage sites. Examples of important
proenzymes include pepsinogen, trypsinogen and chymotrypsinogen, which give
rise to the proteolytic digestive enzymes. Likewise, many of the proteins
involved in the cascade of chemical reactions responsible for blood clotting
are synthesized as proenzymes. Other important proteins, such as peptide
hormones and collagen, are also derived by covalent modification of precursors.</p>

<p>Another mechanism of regulating enzyme activity is to
sequester enzymes in compartments where access to their substrates is limited.
For example, the proteolysis of cell proteins and glycolipids by enzymes
responsible for their degradation is controlled by sequestering these enzymes
within the lysosome.</p>

<p>In contrast to regulatory mechanisms that alter
enzyme concentration, there is an important group of regulatory mechanisms that
do not affect enzyme concentration, are reversible and rapid in action, and
actually carry out most of the moment to moment physiological regulation of
enzyme activity. These mechanisms include allosteric regulation, regulation by
reversible covalent modification and regulation by control proteins such as
calmodulin. Reversible covalent modification is a major mechanism for the rapid
and transient regulation of enzyme activity. The best examples, again, come
from studies on the regulation of glycogen metabolism where phosphorylation of
glycogen synthase and glycogen phosphorylase kinase results in the stimulation
of glycogen degradation while glycogen synthesis is coordinately inhibited.
Numerous other enzymes of intermediary metabolism are affected by
phosphorylation, either positively or negatively. These covalent
phosphorylations can be reversed by a separate sub-subclass of enzymes known as
phosphatases. Recent research has indicated that the aberrant phosphorylation
of growth factor and hormone receptors, as well as of proteins that regulate
cell division, often leads to unregulated cell growth or cancer. The usual
sites for phosphate addition to proteins are the serine, threonine and tyrosine
R group hydroxyl residues.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />

<h1><a name="allosteric"></a>Allosteric Enzymes</h1>

<p>In addition to simple enzymes that interact only with
substrates and inhibitors, there is a class of enzymes that bind small,
physiologically important molecules and modulate activity in ways other than
those described above. These are known as <strong>allosteric enzymes</strong>; 
the small regulatory molecules to which they bind are known as effectors. Allosteric effectors bring about catalytic
modification by binding to the enzyme at distinct allosteric sites, well
removed from the catalytic site, and causing conformational changes that are
transmitted through the bulk of the protein to the catalytically active
site(s).</p>

<p>The hallmark of effectors is that when they bind to
enzymes, they alter the catalytic properties of an enzyme's active site. Those
that increase catalytic activity are known as positive effectors. Effectors
that reduce or inhibit catalytic activity are negative effectors.</p>

<p>Most allosteric enzymes are oligomeric (consisting of
multiple subunits); generally they are located at or near branch points in
metabolic pathways, where they are influential in directing substrates along
one or another of the available metabolic paths. The effectors that modulate
the activity of these allosteric enzymes are of two types. Those activating and
inhibiting effectors that bind at allosteric sites are called heterotropic
effectors. (Thus there exist both
positive and negative heterotropic effectors.) These effectors can assume a
vast diversity of chemical forms, ranging from simple inorganic molecules to
complex nucleotides such as cyclic adenosine monophosphate (cAMP). Their single
defining feature is that they are not identical to the substrate.</p>

<p>In many cases the substrate itself induces distant
allosteric effects when it binds to the catalytic site. Substrates acting as
effectors are said to be homotropic effectors. When the substrate is the effector, it can act as such, either by
binding to the substrate-binding site, or to an allosteric effector site. When
the substrate binds to the catalytic site it transmits an activity-modulating
effect to other subunits of the molecule. Often used as the model of a
homotropic effector is hemoglobin, although it is not a branch-point enzyme and
thus does not fit the definition on all counts.</p>

<p>There are two ways that enzymatic activity can be
altered by effectors: the V<sub>max</sub> can be increased or decreased, or the
K<sub>m</sub> can be raised or lowered. Enzymes whose K<sub>m</sub> is altered
by effectors are said to be K-type enzymes
and the effector a K-type effector. If V<sub>max</sub> is altered, the enzyme
and effector are said to be V-type. Many allosteric enzymes respond to multiple
effectors with V-type and K-type behavior. Here again, hemoglobin is often used
as a model to study allosteric interactions, although it is not strictly an
enzyme.</p>

<p>In the preceding discussion we assumed that allosteric sites and catalytic sites
were homogeneously present on every subunit of an allosteric enzyme. While this
is often the case, there is another class of allosteric enzymes that are
comprised of separate catalytic and regulatory subunits. The archetype of this
class of enzymes is cAMP-dependent protein kinase (PKA), whose mechanism of
activation is illustrated in the Figure below. The enzyme is tetrameric,
containing two catalytic subunits and two regulatory subunits, and
enzymatically inactive. When intracellular cAMP levels rise, one molecule of
cAMP binds to each regulatory subunit, causing the tetramer to dissociate into
one regulatory dimer and two catalytic monomers. In the dissociated form, the
catalytic subunits are fully active; they catalyze the phosphorylation of a
number of other enzymes, such as those involved in regulating glycogen
metabolism. The regulatory subunits have no catalytic activity.</p>

<p class="para1b">
  <img src="/images/pkacamp.jpg" alt="Receptor-mediated regulation of PKA activity" title="Receptor-mediated regulation of PKA activity" />  </p>

  <p class="ridge">Representative pathway for
  the activation of cAMP-dependent protein kinase (PKA). In this example
  glucagon binds to its' cell-surface receptor, thereby activating the
  receptor. Activation of the receptor is coupled to the activation of a
  receptor-coupled G-protein (GTP-binding and hydrolyzing protein) composed of
  3 subunits. Upon activation the &#945;-subunit
  dissociates and binds to and activates adenylate cyclase. Adenylate cylcase then
  converts ATP to cyclic-AMP (cAMP). The cAMP thus produced then binds to the
  regulatory subunits of PKA leading to dissociation of the associated
  catalytic subunits. The catalytic subunits are inactive until dissociated
  from the regulatory subunits. Once released the catalytic subunits of PKA
  phosphorylate numerous substrate using ATP as the phosphate donor.</p>

<br />
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>
<hr />

<h1><a name="clinical"></a>Enzymes in the Diagnosis of Pathology</h1>

<p>The measurement of the serum levels of numerous enzymes has been shown to be of diagnostic significance. 
This is because the presence of these enzymes in the serum indicates that tissue or cellular damage has 
occurred resulting in the release of intracellular components into the blood. Hence, when a physician 
indicates that he/she is going to assay for liver enzymes, the purpose is to ascertain the potential 
for liver cell damage. Commonly assayed enzymes in the blood are the amino transferases: 
alanine transaminase, ALT (sometimes still referred to as serum glutamate-pyruvate aminotransferase, SGPT) 
and aspartate aminotransferase, AST (also referred to as serum glutamate-oxaloacetate aminotransferase, SGOT). 
In addition, the enzymes cardiac troponin I (cTnI), lactate dehydrogenase (LDH) and creatine kinase 
(CK, also called creatine phosphokinase, CPK) are commonly measured in the diagnosis of cardiac infarct. Gamma-glutamyltranferase 
(GGT, &#947;-glutamyltransferase; also called gamma-glutamyl transpeptidase) is involved in glutathione (GSH) 
metablism and also in amino acid transport across membranes. Assessment of GGT levels in the blood is diagnostic 
for diseases of the liver and the biliary system as well as disease of the pancreas.
Other enzymes are assayed under a variety of different clinical situations but they will not be covered here.</p>

<p><strong>Liver Function Enzymes:</strong> The typical liver enzymes measured
are AST and ALT. ALT is particularly diagnostic of liver involvement as this
enzyme is found predominantly in hepatocytes. When assaying for both ALT and
AST the ratio of the level of these two enzymes can also be diagnostic.
Normally in liver disease or damage that is not of viral origin the ratio of
ALT/AST is less than 1. However, with viral hepatitis the ALT/AST ratio will be
greater than 1. Measurement of AST is useful not only for liver involvement but
also for heart disease or damage. The level of AST elevation in the serum is
directly proportional to the number of cells involved as well as on the time
following injury that the AST assay was performed. Following injury, levels of
AST rise within 8 hours and peak 24&ndash;36 hours later. Within 3&ndash;7 days the level
of AST should return to pre-injury levels, provided a continuous insult is not
present or further injury occurs. Although measurement of AST is not, in and of
itself, diagnostic for myocardial infarction, taken together with LDH and CK
measurements (see below) the level of AST is useful for timing of the infarct.</p>

<p><strong>Cardiac Troponins:</strong> Troponins are complexes composed of three regulatory proteins (troponin C, I and T) 
attached to tropomyosin and are found in the grooves between actin filaments in muscle tissue. 
The troponins are found in skeletal and cardiac muscle but not smooth muscle. 
Cardiac troponins found in the serum, specifically troponin I and T, are excellent markers 
for myocardial infarction as well as for any other type of heart muscle damage. 
The measurement of plasma troponin I levels are highly diagnostic of necrosis of cardiac muscle. 
Serum levels of troponin I rise within 4-8 hrs after the onset of chest pains 
caused by myocardial infarction. The levels peak within 12-16 hrs after the onset of 
infarction and return to baseline within 5-9 days. Although measurement of serum LDH 
fractions was once considered the ideal marker for onset and severity of a heart attack, 
the high specificity of troponin I to heart muscle necrosis make this protein the 
preferred marker to measure in patients suspected of suffering a myocardial infarct.</p>

<p><strong>Lactate Dehydrogenases:</strong> The measurement of LDH is especially diagnostic for myocardial infarction because
this enzyme exist in 5 closely related, but slightly different forms
(isozymes). The 5 types and their normal distribution and levels in
non-disease/injury are listed below.</p>

<p class="para2"><strong>LDH 1</strong> &ndash; Found in heart and red-blood 
cells and is 17% &ndash; 27% of the normal serum total.</p>
<p class="para2"><strong>LDH 2</strong> &ndash; Found in heart and red-blood
cells and is 27% &ndash; 37% of the normal serum total.</p>
<p class="para2"><strong>LDH 3</strong> &ndash; Found in a variety of organs
and is 18% &ndash; 25% of the normal serum total.</p>
<p class="para2"><strong>LDH 4</strong> &ndash; Found in a variety of organs
and is 3% &ndash; 8% of the normal serum total. </p>
<p class="para2"><strong>LDH 5</strong> &ndash; Found in liver and skeletal
muscle and is 0% &ndash; 5% of the normal serum total.</p>
<p>Following a myocardial infarct the serum levels of
LDH rise within 24-48 hours reaching a peak by 2&ndash;3 days and return to normal in
5-10 days. Especially diagnostic is a comparison of the LDH-1/LDH-2 ratio.
Normally, this ratio is less than 1. A reversal of this ration is referred to
as a &quot;flipped LDH&quot;. Following an acute myocardial infarct the flipped
LDH ratio will appear in 12&ndash;24 hours and is definitely present by 48 hours in
over 80% of patients. Also important is the fact that persons suffering chest
pain due to angina only will not likely have altered LDH levels.</p>

<p><strong>Creatine Kinases:</strong> Creatine kinase (CK, or creatine phosphokinase, CPK) 
is found primarily in heart and skeletal muscle
as well as the brain. Therefore, measurement of serum CPK levels is a good
diagnostic for injury to these tissues. The levels of CPK will rise within 6
hours of injury and peak by around 18 hours. If the injury is not persistent
the level of CK returns to normal within 2&ndash;3 days. Like LDH, there are
tissue-specific isozymes of CPK and there designations are described below.</p>

<p class="para2"><strong>CPK3</strong>
(CPK-MM) is the predominant isozyme in muscle and is 100% of the normal serum
total.</p>
<p class="para2"><strong>CPK2</strong> (CPK-MB) accounts for about 35%
of the CPK activity in cardiac muscle, but less than 5% in skeletal muscle and
is 0% of the normal serum total.</p>
<p class="para2"><strong>CPK1</strong> (CPK-BB) is the characteristic
isozyme in brain and is in significant amounts in smooth muscle and is 0% of
the normal serum total.</p>
<p>Since most of the released CPK after a myocardial
infarction is CPK-MB, an increased ratio of CPK-MB to total CPK may help in
diagnosis of an acute infarction, but an increase of total CPK in itself may
not. CPK-MB levels rise 3&ndash;6 hours after a myocardial infarct and peak 12&ndash;24
hours later if no further damage occurs and returns to normal 12&ndash;48 hours after
the infarct.</p>

<p><strong>Gamma-Glutamyltransferase: </strong>Normal plasma levels of GGT are 15-85 IU/L for males and 5-55 IU/L for women.
Elevated serum GGT levels are found in diseases of the liver, biliary system, and pancreas. 
Slight elevations in serum GGT levels are also seen in cardiovascular disease, in 
particular, GGT accumulates in atherosclerotic plaques. GGT-containing protein aggregates circulate 
in the blood in certain pathologies such as <a href="/metabolic-syndrome.php">metabolic syndrome</a>, 
alcohol addiction and chronic liver disease.</p>

<p><strong>Carbonic Anhydrases:</strong> Humans express 15 isoforms of carbonic anhydrase with 12 exhibitng functional enzymatic 
activity. Carbonic anhydrases form a large family of genes encoding zinc metalloenzymes of great physiologic importance. As catalysts of the reversible hydration of carbon dioxide, these enzymes participate in a variety of biologic processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid (Dodgson et al., 1991). CAs are encoded by members of 3 independent CA gene families, i.e., alpha-CA, beta-CA, and gamma-CA (Hewett-Emmett and Tashian, 1996). Genes in the alpha-carbonic anhydrase family encode either active carbonic anhydrase isozymes or 'acatalytic' (i.e., devoid of CO2 hydration activity) carbonic anhydrase-related proteins. Alpha-carbonic anhydrases show extensive diversity in tissue distribution and in their putative or established biologic functions. Some of the alpha-CAs are expressed in almost all tissues (e.g., CA II, 611492), whereas some show a more restricted expression (e.g., CA VI (114780) in salivary glands). In cells, they may reside in cytoplasm, in mitochondria, or in secretory granules, or associate with membranes.</p>
<span style="margin-left:8px"><a href="/enzyme-kinetics.php#top">back to the top</a></span>

<hr />
<span style="margin-left:8px"><a href="/index.php">Return to The Medical Biochemistry Page</a></span>

<hr />
<span style="font-size: 10pt"><span style="margin-left:8px">Michael W King, PhD &#124; &#169; 1996&ndash;2013 themedicalbiochemistrypage.org, LLC &#124; info @ 
themedicalbiochemistrypage.org</span></span>

<hr />

<h4>Last modified: August 21, 2013</h4>
</div>

</div>
</body>

</html>
