<base href="http://www.ncbi.nlm.nih.gov/About/primer/phylo.html">
<html lang="en">
<head>
<title>Phylogenetics Factsheet</title>

<link rel=stylesheet href="../aboutncbi.css" type="text/css">

<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="author" content="NCBI_user_services">
<meta name="robots" content="index, nofollow, NOARCHIVE">
<meta name="Description" content="NCBI creates public databases, conducts research in computational biology, develops software tools for analyzing genome data, and disseminates biomedical information">
<meta name="keywords" content="basic introduction to NCBI resources, phylogenetics, taxonomic classification,
systematics, evolutionary process, phlogenetic trees, cladistics, molecular phylogenetics, NCBI taxonomy">

<script language="JavaScript">
<!--
function MM_reloadPage(init) {  //reloads the window if Nav4 resized
  if (init==true) with (navigator) {if ((appName=="Netscape")&&(parseInt(appVersion)==4)) {
    document.MM_pgW=innerWidth; document.MM_pgH=innerHeight; onresize=MM_reloadPage; }}
  else if (innerWidth!=document.MM_pgW || innerHeight!=document.MM_pgH) location.reload();
}
MM_reloadPage(true);
// -->
</script>
</head>

<BODY BACKGROUND="images/sidebar.gif" width="600" BGCOLOR="#FFFFFF" TEXT="#000000" LEFTMARGIN="0" TOPMARGIN="0">
<table width="600" cellspacing="0" cellpadding="5">
  <tr>
	<td width="140" align="center"><a href="http://www.ncbi.nlm.nih.gov"><img src="images/swoosh.gif" alt="NCBI Logo" border="0"></a></td>
    <td width="460"><img src="images/primerbar.gif" alt="A Science Primer Titlebar" width="460" height="60"></td>
  </tr>
 </table>
<table width="600" border="0" cellspacing="0" cellpadding="5" bgcolor="#006699">
     <tr>
       <td width="100%" align="center"><font color="#FFFFFF"><span class="title"><a NAME="phylogenetics">National</a> Center for Biotechnology Information</span></font></td>
	</tr>
	</table>	
<table width="600" border="0" cellspacing="0" cellpadding="0">
	<td>
<table width="130"><tr><td>&nbsp;</td></tr>
	</table>
	</td>
	<td>
<table width="470" border="1" cellspacing="0" cellpadding="0" bgcolor="#FFFFCC" bordercolor="#006699">
		  <tr>
		  <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../index.html">About 
            NCBI</a></span></td>
		  <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../glance/index.html">NCBI 
            at a Glance</a></span></td>
          <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="index.html">A 
            Science Primer</a></span></td>
		  <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../tools/index.html">Databases 
            and Tools</a></span></td>		  
      </tr>
	  <tr>
		  <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../human/index.html">Human 
            Genome Resources</a></span></td>
		  <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../model/index.html">Model 
            Organisms Guide</a></span></td>
          <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../outreach/index.html">Outreach 
            and Education</a></span></td>
          <td width="25%" align="center" bgcolor="#FFFFCC"><span class="nav"><a href="../news/index.html">News</a></span></td>    
	</tr>
  </table>
    </td>
  </tr>
</table>
<br>
<!-- This table contains both the Side bar table and the main content -->
<TABLE VALIGN="TOP" width="600" border="0" cellpadding="0" cellspacing="0">

<!-- Sets off the cell which contains the side bar links table -->
<TD VALIGN="TOP" WIDTH="130"> 
<!-- This table contains the sidebar links --> 
      <table width="130" border="0" cellspacing="5" cellpadding="0">
	  <tr> 
          <td><a href="../sitemap.html" class="sitemap"><span class="sitemap">About NCBI<BR>Site Map</span></a></td>
        </tr>
        <tr> 
          <td><br><span class="title">Science Primer:</span></td>
        </tr>
		<tr> 
          <td><br><a href="bioinformatics.html" class="sidebar"><span class="sidebar">Bioinformatics</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="mapping.html" class="sidebar"><span class="sidebar">Genome 
            Mapping</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="molecularmod.html" class="sidebar"><span class="sidebar">Molecular 
            Modeling</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="snps.html" class="sidebar"><span class="sidebar">SNPs</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="est.html" class="sidebar"><span class="sidebar">ESTs</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="microarrays.html" class="sidebar"><span class="sidebar">Microarray 
            Technology</span></a></td>
        </tr>
        
		
		<tr> 
          <td><br><a href="genetics.html" class="sidebar"><span class="sidebar">Molecular Genetics</span></a></td>
        </tr>
        <tr> 
          <td><br><a href="pharm.html" class="sidebar"><span class="sidebar">Pharmacogenomics</span></a></td>
        </tr>       
		
      </table>
	  

<!--    spacer column, keeps text off background    -->
    <td width="10">&nbsp;</td>

<!-- This table contains the main body information -->  
<TD valign="top" width="470"> 

      <table width="470" border="0" cellspacing="5" cellpadding="0">
        <TBODY>
        <TR>
          <TD align="center"><br>&nbsp;<br><H2><font color="#808080">Just the Facts: A Basic Introduction to the Science Underlying NCBI Resources</font></H2></TD>
			</TR>
  </table>
  
      <table width="470" border="0" cellspacing="5" cellpadding="0">
        <tr> 
          <td align="center"><br>
            <h3>SYSTEMATICS AND MOLECULAR PHYLOGENETICS<br>
            </h3>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>Classifying Organisms</h3>
            <p>Have you ever noticed that when you see an insect or a bird, there 
              is real satisfaction in giving it a name, and an uncomfortable uncertainty 
              when you can't? Along these same lines, consider the bewildering 
              number and variety of organisms that live, or have lived, on this 
              earth. If we did not know what to call these organisms, how could 
              we communicate ideas about them, let alone the history of life? Thanks 
			  to <b>taxonomy</b>, the field of science that classifies life into groups, 
			  we can discuss just about any organism, from bacteria to man.</p>
            <p>Carolus Linnaeus pioneered the grouping of organisms based on scientific 
              names using Latin. His system of giving an organism a scientific 
              name of two parts, sometimes more, is called <b>binomial nomenclature</b>, 
              or "two-word naming". His scheme was based on physical similarities 
              and differences, referred to as <b>characters</b>. Today, taxonomic 
              classification is much more complex and takes into account cellular 
              types and organization, biochemical similarities, and genetic similarities. 
              Taxonomy is but one aspect of a much larger field called systematics.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table width="100%" border="2" cellspacing="0" cellpadding="5" bgcolor="#CCFFCC" bordercolor="#006699">
              <tr> 
                <td align="center" bgcolor="#006699"><span class="title"><font color="#FFFFFF">Taxonomic 
                  Classification</font></span></td>
              </tr>
              <tr> 
                <td><br>
                  <span class="inset">Taxonomic ranks approximate evolutionary distances
				  among groups of organisms.  For example, species belonging to two 
				  different superkingdoms are most distantly related (their common 
				  ancestor diverged in the distant past), with progressively more 
				  exclusive groups indicated by phylum, class and so on, down to 
				  infraspecific ranks, or ranks occurring within a species.  
				  Infraspecific ranks, such as subspecies, varietas, and forma, 
				  denote the closest evolutionary relationship.  See the 
				  simplified classification of humans below.</span><br><br>
                  <span class="inset"><b>Taxonomists</b>, scientists who classify 
                  living organisms, define a species as any group of closely related 
				  organisms that can produce fertile offspring. Two organisms are 
				  more closely "related" as they approach the level of species, 
				  that is, they have more genes in common. The level of species 
				  can be further divided into smaller segments. A <b>population</b> 
				  is the smallest unit of a species and is made up of organisms of 
				  the same species. Sometimes, a population will physically alter 
				  over time to suit the needs of its environment. This is called a 
				  <b>cline</b> and can make members of the same species look different.</span><br>
                  <br>
                  <table width="75%" align="center" border="2" cellspacing="0" cellpadding="10" bordercolor="#006699" bgcolor="#FFFFCC">
                    <td align="center"> 
                      <p><b>Taxonomic Classification of Man</b><br>
                        <i>Homo sapiens</i></p>
						<table align="center" width="80%" border="0">
                        <tr> 
						<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
				        <td> 
                            <span class="inset">&nbsp;Superkingdom:&nbsp;Eukaryota</span>
                          </td>
                         </tr>
                        <tr> 
      					<td>&nbsp;&nbsp;</td>
		        		<td>&nbsp;&nbsp;</td>
                          <td> 
                           <span class="inset">&nbsp;&nbsp;&nbsp;Kingdom:&nbsp;Metazoa</span>
                          </td>
                          
                        </tr>
                        <tr> 
                		<td>&nbsp;&nbsp;</td>
		        		<td>&nbsp;&nbsp;</td>
                          <td> 
                            <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Phylum:&nbsp;Chordata</span>
                          </td>
                       </tr>
                        <tr> 
                  		<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
                          <td> 
                            <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Class:&nbsp;Mammalia</span>
                          </td>
                        </tr>
                        <tr> 
                    	<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
                          <td> 
                           <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Order:&nbsp;Primata</span>
                          </td>
                        </tr>
                          <tr> 
						<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
						  <td>
                            <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Family:&nbsp;Hominidae</span>
                          </td>
                        </tr>
                        <tr> 
                   		<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
                          <td> 
                            <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Genus:&nbsp;Homo</span>
                          </tr>
					   <tr> 
       					<td>&nbsp;&nbsp;</td>
						<td>&nbsp;&nbsp;</td>
					   <td> 
                            <span class="inset">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Species:&nbsp;sapiens</span>
				
                          </td>
                        </tr>
                      </table>
                    </td>
                  </table>
                </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>What Is Phylogenetic Systematics?</h3>
            <p>Carolus Linnaeus was also credited with pioneering <b>systematics</b>, the 
              field of science dealing with the diversity of life and the relationship 
              between life's components. Systematics reaches beyond taxonomy to 
              elucidate new methods and theories that can be used to classify 
              species based on similarity of traits and possible mechanisms of 
              <b>evolution</b>, a change in the gene pool of a population over 
              time.</p>
            <p><b>Phylogenetic systematics</b> is that field of biology that 
              does deal with identifying and understanding the evolutionary 
              relationships among the many different kinds of life on earth, 
              both living (<b>extant</b>) and dead (<b>extinct</b>). Evolutionary 
			  theory states that similarity among individuals or species is attributable
			  to common descent, or inheritance from a common ancestor.  Thus, 
              the relationships established by phylogenetic systematics often 
              describe a species' evolutionary history and, hence, its <b>phylogeny</b>,
			  the historical relationships among lineages or organisms or their parts, 
              such as their genes. 
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="90%" border="2" cellspacing="0" cellpadding="5" bgcolor="#FFFFCC" bordercolor="#006699">
              <td> <span class="inset">Charles Darwin was the first to recognize that the 
			  systematic hierarchy represented a rough approximation of evolutionary 
			  history.  However, it was not until the 1950s that the German entomologist 
			  Willi Hennig proposed that systematics should reflect the known evolutionary
			  history of lineages as closely as possible, an approach he called
			  <b>phylogenetic systematics</b>.  The followers of Hennig were disparagingly
			  referred to as &quot;cladists&quot; by his opponents, because of the emphasis on
			  recognizing only <b>monophyletic groups</b>, a group plus all of its descendents,
			  or clades. However, the cladists quickly adopted that term as a helpful label, 
			  and nowadays, cladistic approaches to systematics are used routinely.</span> </td>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>Understanding the Evolutionary Process</h3>
            <h4>Genetic Variation: Changes in a Gene Pool</h4>
            <p>Evolution is not always discrete with clearly defined boundaries 
              that pinpoint the origin of a new species, nor is it a steady continuum. 
              Evolution requires genetic variation which results 
              from changes within a <b>gene pool</b>, the genetic make-up of a 
              specific population. A gene pool is the combination of all the <b>alleles</b>
			  &mdash;alternative forms of a genetic locus&mdash;for all traits that population 
			  may exhibit. Changes in a gene pool can result from <b>mutation</b>&mdash;variation 
              within a particular gene&mdash;or from changes in <b>gene frequency</b>&mdash;the 
			  proportion of an allele in a given population.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>How Does Genetic Variation Occur?</h4>
            <p>Every organism possesses a <b>genome</b> that contains all of the biological 
              information needed to construct and maintain a living example of 
              that organism. The biological information contained in a genome 
              is encoded in the nucleotide sequence of its DNA or RNA molecules 
              and is divided into discrete units called <b>genes</b>. The information 
              stored in a gene is read by proteins, which attach to the genome 
              and initiate a series of reactions called <b>gene expression</b>. 
            </p>
            <p>Every time a cell divides, it must make a complete copy of its 
              genome, a process called <b>DNA replication</b>. DNA replication 
              must be extremely accurate to avoid introducing <b>mutations</b>, 
              or changes in the nucleotide sequence of a short region of the genome. 
              Inevitably, some mutations do occur, usually in one of two ways; 
              either from errors in DNA replication or from damaging effects of 
              chemical agents or radiation that react with DNA and change the 
              structure of individual nucleotides. Many of these mutations result 
              in a change that has no effect on the functioning of the genome, 
              referred to as <b>silent mutations</b>. Silent mutations include 
              virtually all changes that happen in the non-coding components of 
              genes and gene-related sequences.</p>
            <p>Mutations in the coding regions of genes are much more important. 
              Here we must consider the importance of the same mutation in a <b>somatic 
              cell</b> compared with a <b>germ line cell</b>. A somatic cell is 
              any cell of an organism other than a reproductive cell, such as a 
              sperm or egg cell. A germ cell line is any line of cells that gives 
              rise to gametes and is continuous through the generations. Because 
              a somatic cell does not pass on copies of its genome to the next 
              generation, a somatic cell mutation is important only for the organism 
              in which it occurs and has no potential evolutionary impact. In 
              fact, most somatic mutations have no significant effect because 
              there are many other identical cells in the same tissue. </p>
            <p>On the other hand, mutations in germ cells can be transmitted to 
              the next generation and will then be present in all of the cells of 
              an individual who inherits that mutation. Even still, mutations 
              within germ line cells may not change the phenotype of the organism 
              in any significant way. Those mutations that do have an evolutionary 
              effect can be divided into two categories,<b> loss-of-function mutations</b> 
              and <b>gain-of-function mutations</b>. A loss-of-function mutation 
              results in reduced or abolished protein function. Gain-of-function 
              mutations, which are much less common, confer an abnormal activity 
              on a protein.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="90%" cellspacing="0" cellpadding="5" border="2" bgcolor="#FFFFCC" bordercolor="#006699">
              <td><span class="inset">The randomness with which mutations can 
                occur is an important concept in biology and is a requirement 
                of the Darwinian view of evolution, which holds that changes in 
                the characteristics of an organism occur by chance and are not 
                influenced by the environment in which the organism lives. Beneficial 
                changes within an organism are then positively selected for, whereas 
                harmful changes are negatively selected.</span></td>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>The Drivers of Evolution: Selection, Drift, and Founder Effects</h4>
            <p>We just discussed that new alleles appear in a population because 
              of mutations that occur in the reproductive cells of an organism. 
              This means that many genes are <b>polymorphic</b>, that is, two 
              or more alleles for that gene are present in a population. Each 
              of these alleles has its own <b>allele</b> or <b>gene frequency</b>, a 
              measure of how common an allele is in a population. Allele frequencies 
              vary over time because of two conditions, natural selection and random 
              drift. </p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Natural Selection</i></h4>
            <p><b>Natural selection</b> is the process whereby one <b>genotype</b>, the 
              hereditary constitution of an individual, leaves more offspring 
              than another genotype because of superior life attributes, termed 
              <b>fitness</b>. Natural selection acts on genetic variation by conferring 
              a survival advantage to those individuals harboring a particular 
              mutation that tends to favor a changing environmental condition. 
              These individuals then reproduce and pass on this "new" gene, altering 
              their gene pool. Natural selection, therefore, decreases the frequencies 
              of alleles that reduce the fitness of an organism and increases the 
              frequency of alleles that improve fitness.</p>
            <table align="center" width="90%" border="2" cellspacing="0" cellpadding="5" bgcolor="#FFFFCC" bordercolor="#006699">
              <td> 
                <table align="center" width="100%">
                  <tr> 
                    <td> 
                      <span class="inset">&quot;<b>Natural Selection</b>&quot; is the principle 
                        by which each slight variation, if useful, is preserved.</span>
                    </td>
                  </tr>
                  <tr> 
                    <td align="right"> 
                      <span class="inset"><i>Charles Darwin</i></span>
                    </td>
                  </tr>
                </table>
              </td>
            </table>
            <br>
            <p>It is important to point out that natural selection does not always 
              represent progress, only adaptation to a changing surrounding, that 
              is, evolution attributable to natural selection is devoid of intent&mdash;
			  something does not evolve to better itself, only to adapt. 
			  Because environments are always changing, what was once an advantageous 
			  mutation can often become a liability further down the evolutionary line.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Random Drift</i></h4>
            <p>The term <b>random drift</b> actually encompasses a number of 
              distinct processes, sometimes referred to as outcomes. They include 
              <b>indiscriminate parent sampling</b>, the <b>founder effect</b>, 
              and fluctuations in the rate of evolutionary processes such as selection, 
              migration, and mutation. <b>Parent sampling</b> is the process of 
              determining which organisms of one generation will be the parents 
              of the next generation. Parent sampling may be <b>discriminate</b>, 
              that is, with regard to fitness differences, or <b>indiscriminate</b>, without 
              regard to fitness differences. Discriminate parent sampling is generally 
              considered natural selection, whereas indiscriminate parent sampling 
              is considered random drift.</p>
            <table align="center" width="95%" cellspacing="0" cellpadding="5" border="2" bgcolor="#FFFFCC" bordercolor="#006699">
              <td align="center" bgcolor="#006699"><span class="title"><font color="#FFFFFF">What 
                Is Sampling?</font></span></td>
              </tr>
              <tr> 
                <td> 
                  <span class="inset">Suppose a population of red and brown 
                    squirrels share a habitat with a color blind predator. 
					Although the predator is color blind, the brown squirrels seem 
                    to die in greater numbers than the red squirrels, suggesting 
                    that the brown squirrels just seem to be unlucky enough to 
                    come into contact with the predator more often. As a result, 
                    the frequency of brown squirrels in the next generation is 
                    reduced. More red squirrels survive to reproduce, or are sampled, 
                    but it is without regard to any differences in fitness between 
                    the two groups. The physical differences of the groups do 
                    not play a causal role in the differences in reproductive 
                    success.</span><br><br>
                  <span class="inset">Now, lets say that the predator is not 
                    color blind and can now see the red squirrels better than the 
                    brown squirrels, resulting in a better survival rate for the 
                    brown squirrels. This would be a case of discriminate parent 
                    sampling, or natural selection.</span>
                </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Founder Effect</i></h4>
            <p>Another important cause of genetic drift is the <b>founder effect</b>, the 
              difference between the gene pool of a population as a whole and 
              that of a newly isolated population of the same species. The founder 
              effect occurs when populations are started from a small number of 
              pioneer individuals of one original population. Because of small sample 
              size, the new population could have a much different genetic ratio 
              than the original population. An example of the founder effect would 
              be when a plant population results from a single seed.</p>
            <p>Thus far, we have discussed natural selection and random drift as 
              events that occur in isolation from one another. However, in most 
              populations, the two processes will be occurring at the same time. 
              Furthermore, there is great debate over whether, in particular instances 
              and in general, natural selection is more prevalent that random 
              drift.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>Phylogenetic Trees: Presenting Evolutionary Relationships</h3>
            <p>Systematics describes the pattern of relationships among taxa and 
              is intended to help us understand the history of all life. But history 
              is not something we can see&mdash;it has happened once and leaves only 
              clues as to the actual events. Scientists use these clues to build 
              hypotheses, or models, of life's history. In phylogenetic studies, 
              the most convenient way of visually presenting evolutionary relationships 
              among a group of organisms is through illustrations called <b>phylogenetic 
              trees</b>. </p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="right" cellpadding="5" cellspacing="0" border="0">
              <td><img src="images/treechart.gif" width="250" height="157" alt="A phylogenetic tree, as described to the left."></td>
            </table>
            <span class="inset"> 
            <ul>
              <li><b>Node:</b> represents a taxonomic unit. This can be either 
                an existing species or an ancestor.</li>
              <br>
              <li><b>Branch:</b> defines the relationship between the taxa in 
                terms of descent and ancestry.</li>
              <br>
              <li><b>Topology</b>: the branching patterns of the tree.</li>
              <br>
              <li><b>Branch length</b>: represents the number of changes that 
                have occurred in the branch.</li>
              <br>
              <li><b>Root</b>: the common ancestor of all taxa.</li>
              <br>
              <li><b>Distance scale</b>: scale that represents the number of differences 
                between organisms or sequences.</li>
              <br>
              <li><b>Clade</b>: a group of two or more taxa or DNA sequences 
                that includes both their common ancestor and all of their descendents.</li>
              <br>
              <li><b>Operational Taxonomic Unit (OTU)</b>: taxonomic level of sampling 
			  selected by the user to be used in a study, such as individuals, populations,
			  species, genera, or bacterial strains.</li>
            </ul>
            </span> </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <p>A phylogenetic tree is composed of <b>nodes</b>, each representing 
              a taxonomic unit (species, populations, individuals), and <b>branches</b>, 
              which define the relationship between the taxonomic units in terms 
              of descent and ancestry. Only one branch can connect any two adjacent 
              nodes. The branching pattern of the tree is called the <b>topology</b>, 
              and the branch length usually represents the number of changes that 
              have occurred in the branch. This is called a <b>scaled branch</b>. 
              Scaled trees are often calibrated to represent the passage of time. 
			  Such trees have a theoretical basis in the particular gene or genes
			  under analysis.  Branches can also be <b>unscaled</b>, which means that the 
              branch length is not proportional to the number of changes that 
              has occurred, although the actual number may be indicated numerically 
              somewhere on the branch. Phylogenetic trees may also be either <b>rooted</b> 
              or <b>unrooted</b>. In rooted trees, there is a particular node, 
              called the <b>root</b>, representing a common ancestor, from which 
              a unique path leads to any other node. An unrooted tree only specifies 
              the relationship among species, without identifying a common ancestor, 
              or evolutionary path.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table width="100%" cellspacing="0" cellpadding="2" border="0">
              <td align=center> <img src="images/trees3.gif" width="425" height="470" alt="Figure 1.  Possible Ways of Drawing a Tree"></td>
            </table>
          </td>
        </tr>
        <tr> 
          <td><br>
            <p><b>Figure 1.</b> Possible ways of drawing a tree. <br><br>
            <span class="inset">Phylogenetic trees, a convenient way of representing 
            evolutionary relationships among a group of organisms, can be drawn 
            in various ways. Branches on phylogenetic trees may be scaled (<i>top 
            panel</i>) representing the amount of evolutionary change, time, or both, when 
            there is a molecular clock, or they may be unscaled (<i>middle panel</i>) 
            and have no direct correspondence with either time or amount of evolutionary 
            change. Phylogenetic trees may be rooted (<i>top</i> and <i>middle panels</i>) or 
            unrooted (<i>bottom panels</i>). In the case of unrooted trees, branching 
            relationships between taxa are specified by the way they are connected 
            to each other, but the position of the common ancestor is not. For 
            example, on an unrooted tree with five species, there are five branches 
            (four external, one internal) on which the tree can be rooted. Rooting 
            on each of the five branches has different implications for evolutionary 
            relationships.</span></td></tr>. 
			<tr>
			<td align=left> <span class=credit>Text and figures adapted with permission 
			from A. Vierstraete, University of Ghent, Belgium.</span>
			 </td>
			</tr>
            
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td> 
            <h3>Methods of Phylogenetic Analysis</h3>
            <p>Two major groups of analyses exist to examine phylogenetic relationships: 
              <b>phenetic methods</b> and <b>cladistic methods</b>. It is important 
              to note that phenetics and cladistics have had an uneasy relationship 
              over the last 40 years or so. Most of today's evolutionary biologists 
              favor cladistics, although a strictly cladistic approach may 
              result in counterintuitive results. </p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Phenetic Method of Analysis</i></h4>
            <p>Phenetics, also known as <b>numerical taxonomy</b>, involves the 
              use of various measures of overall similarity for the ranking of 
              species. There is no restriction on the number or type of characters 
              (data) that can be used, although all data must be first converted 
              to a numerical value, without any character "weighting". Each organism 
              is then compared with every other for all characters measured, and 
              the number of similarities (or differences) is calculated. The organisms 
              are then clustered in such a way that the most similar are grouped 
              close together and the more different ones are linked more distantly. 
              The taxonomic clusters, called <b>phenograms</b>, that result from 
              such an analysis do not necessarily reflect genetic similarity or 
              evolutionary relatedness. The lack of evolutionary significance 
              in phenetics has meant that this system has had little impact on 
              animal classification, and as a consequence, interest in and use 
              of phenetics has been declining in recent years.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Cladistic Method of Analysis</i></h4>
            <p>An alternative approach to diagramming relationships between taxa 
              is called <b>cladistics</b>. The basic assumption behind cladistics 
              is that members of a group share a common evolutionary history. 
              Thus, they are more closely related to one another than they are 
              to other groups of organisms. Related groups of organisms are recognized 
              because they share a set of unique features (<b>apomorphies</b>) 
              that were not present in distant ancestors but which are shared 
              by most or all of the organisms within the group. These shared derived 
              characteristics are called <b>synapomorphies</b>. Therefore, in 
              contrast to phenetics, cladistics groupings do not depend on whether 
              organisms share physical traits but depend on their evolutionary relationships. 
              Indeed, in cladistic analyses two organisms may share numerous characteristics 
              but still be considered members of different groups.</p>
            <p>Cladistic analysis entails a number of assumptions.  For example, species
			  are assumed to arise primarily by <b>bifurcation</b>, or separation, of the 
			  ancestral lineage; species are often considered to become extinct upon 
			  hybridization (crossbreeding); and hybridization is assumed to be rare or 
			  absent.  In addition, cladistic groupings must possess the following characteristics: 
              all species in a grouping must share a common ancestor and all 
              species derived from a common ancestor must be included in the taxon. 
              The application of these requirements results in the following terms 
              being used to describe the different ways in which groupings can 
              be made:</p>
            <span class="li"> 
            <ul>
              <li>A <b>monophyletic grouping</b> is one in which all species share 
                a common ancestor, and all species derived from that common ancestor 
                are included. This is the only form of grouping accepted as valid 
                by cladists.</li>
              <li>A <b>paraphyletic grouping</b> is one in which all species share 
                a common ancestor, but not all species derived from that common 
                ancestor are included.</li>
              <li>A <b>polyphyletic grouping</b> is one in which species that 
                do not share an immediate common ancestor are lumped together, 
                while excluding other members that would link them.</li>
            </ul>
            </span> </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>The Origins of Molecular Phylogenetics</h3>
            <p><b>Macromolecular data</b>, meaning gene (DNA) and protein sequences, 
              are accumulating at an increasing rate because of recent advances in 
              molecular biology. For the evolutionary biologist, the rapid accumulation 
              of sequence data from whole genomes has been a major advance, because
              the very nature of DNA allows it to be used as a "document" of evolutionary 
              history. Comparisons of the DNA sequences of various genes between 
              different organisms can tell a scientist a lot about the relationships 
              of organisms that cannot otherwise be inferred from <b>morphology</b>, 
              or an organism's outer form and inner structure. Because genomes evolve 
              by the gradual accumulation of mutations, the amount of nucleotide 
              sequence difference between a pair of genomes from different organisms 
              should indicate how recently those two genomes shared a common ancestor. 
              Two genomes that diverged in the recent past should have fewer differences 
              than two genomes whose common ancestor is more ancient. Therefore, 
              by comparing different genomes with each other, it should be possible 
              to derive evolutionary relationships between them, the major objective 
              of molecular phylogenetics.</p>
            <p><b>Molecular phylogenetics</b> attempts to determine the rates 
              and patterns of change occurring in DNA and proteins and to reconstruct 
              the evolutionary history of genes and organisms. Two general approaches 
              may be taken to obtain this information. In the first approach, 
              scientists use DNA to study the evolution of an organism. In the 
              second approach, different organisms are used to study the evolution 
              of DNA. Whatever the approach, the general goal is to <b>infer process 
              from pattern</b>: the processes of organismal evolution deduced 
              from patterns of DNA variation and processes of molecular evolution 
              inferred from the patterns of variations in the DNA itself.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>Molecular Phylogenetic Analysis: Fundamental Elements</h4>
            <table align="right" border="0" cellspacing="0" cellpadding="5" width="150" bgcolor="#FFFFFF">
              <tr> 
                <td> 
                  <table border="2" cellpadding="6" cellspacing="0" width="100%" bgcolor="#FFFFCC" bordercolor="#006699">
                    <tr> 
                      <td width="100%"><span class="inset">Nucleotide and protein 
                        sequences can also be used to generate trees. DNA, RNA, 
                        and protein sequences can be considered as phenotypic 
                        traits. The sequences depict the relationship of genes 
                        and usually of the organism in which the genes are found.</span></td>
                    </tr>
                  </table>
                </td>
              </tr>
            </table>
            <p>As we just discussed, macromolecules, especially gene and protein 
              sequences, have surpassed morphological and other organismal characters 
              as the most popular forms of data for phylogenetic analyses. Therefore, 
              this next section will concentrate only on molecular data.</p>
            <p>It is important to point out that a single, all-purpose 
              recipe does not exist for phylogenetic analysis of molecular 
              data. Although numerous algorithms, procedures, and computer programs 
              have been developed, their reliability and practicality are, in 
              all cases, dependent upon the size and structure of the dataset 
              under analysis. The merits and shortfalls of these various methods 
              are subject to much scientific debate, because the danger of generating 
              incorrect results is greater in computational molecular phylogenetics 
              than in many other fields of science. Occasionally, the limiting 
              factor in such analyses is not so much the computational method 
              used, but the users' understanding of what the method is actually 
              doing with the data. Therefore, the goal of this section is to demonstrate 
              to the reader that practical analysis should be thought of both 
              as a search for a correct model (analysis) as well as a search for 
              the correct tree (outcome).</p>
            <p>Phylogenetic tree-building models presume particular evolutionary 
              models. For any given set of data, these models may be violated 
              because of various occurrences, such as the transfer of genetic 
              material between organisms. Therefore, when interpreting a given 
              analysis, a person should always consider the model used and entertain 
              possible explanations for the results obtained. For example, models 
              used in molecular phylogenetic analysis methods make "default" assumptions, 
              including:</p>
            <span class="li"> 
            <ul>
              <li>The sequence is correct and originates from the specified source.</li><br>
              <li>The sequences are homologous&mdash;all descended in some way 
                from a shared ancestral sequence.</li><br>
              <li>Each position in a sequence alignment is homologous with every 
                other in that alignment.</li><br>
              <li>Each of the multiple sequences included in a common analysis 
                has a common phylogenetic history with the other sequences.</li><br>
              <li>The sampling of taxa is adequate to resolve the problem under 
                study.</li><br>
              <li>Sequence variation among the samples is representative of the 
                broader group.</li><br>
              <li>The sequence variability in the sample contains phylogenetic 
                signal adequate to resolve the problem under study. </li><br>
            </ul>
            </span> </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="95%" cellspacing="0" cellpadding="5" border="2" bgcolor="#FFFFCC" bordercolor="#006699">
              <td align="center" bgcolor="#006699"><span class="title"><font color="#FFFFFF">The 
                Four Steps of Phylogenetic Analysis</font></span></td>
              </tr>
              <tr> 
                <td> 
                  <p> 
                  <span class="inset">A straightforward phylogenetic analysis 
                  consists of four steps: 
                  <ol>
                    <li>Alignment&mdash;building the data model and extracting a dataset.</li><br><br>
                    <li>Determining the substitution model&mdash;consider sequence 
                      variation.</li><br><br>
                    <li>Tree building.</li><br><br>
                    <li>Tree evaluation.</li><br>
                  </ol>
                  </span> </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>Tree Building: Key Features of DNA-based Phylogenetic Trees</h4>
            <p>Studies of gene and protein evolution often involve the comparison 
              of <b>homologs</b>, sequences that have common origins but may or 
              may not have common activity. Sequences that share an arbitrary 
              level of similarity determined by alignment of matching bases are 
              <b>homologous</b>. These sequences are inherited from a common 
              ancestor that possessed similar structure, although the ancestor 
              may be difficult to determine because it has been modified through 
              descent. </p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" border="0" cellspacing="0" cellpadding="5" width="95%" bgcolor="#FFFFFF">
              <tr> 
                <td> 
                  <table border="2" cellpadding="6" cellspacing="0" width="100%" bgcolor="#FFFFCC" bordercolor="#006699">
                    <tr> 
                      <td width="100%"> 
                        <p>Homologs are most commonly defined as orthologs, paralogs, 
                          or xenologs.</p>
                        <span class="inset"> 
                        <ol>
                          <b> Orthologs</b> are homologs produced by speciation&mdash;they 
                          represent genes derived from a common ancestor that 
                          diverged because of divergence of the organism. Orthologs 
                          tend to have similar function. 
                        </ol>
                        </span> <span class="inset">
                        <ol>
                          <b> Paralogs</b> are homologs produced by gene duplication 
                          and represent genes derived from a common ancestral 
                          gene that duplicated within an organism and then diverged. 
                          Paralogs tend to have different functions. 
                        </ol>
                        </span> <span class="inset">
                        <ol>
                          <b> Xenologs</b> are homologs resulting from the horizontal 
                          transfer of a gene between two organisms. The function 
                          of xenologs can be variable, depending on how significant 
                          the change in context was for the horizontally moving 
                          gene. In general, though, the function tends to be similar. 
                        </ol>
                        </span></td>
                    </tr>
                  </table>
                </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td><p>A typical gene-based phylogenetic tree is depicted below. This tree 
            shows the relationship between four homologous genes: A, B, C, and 
            D. The topology of this tree consists of four external nodes (<b><font color="#FF0000">A</font>, 
			<font color="#FF0000">B</font>, <font color="#FF0000">C</font>,</b> 
            and <b> <font color="#FF0000">D</font></b>), each one representing 
            one of the four genes, and two internal nodes (<b><font color="#0000FF">e</font></b> 
			and <b><font color="#0000FF">f</font></b>) representing 
            ancestral genes. The branch lengths indicate the degree of evolutionary 
            differences between the genes. This particular tree is unrooted&mdash;it 
            is only an illustration of the relationships between genes A, B, C, 
            and D and does not signify anything about the series of evolutionary 
            events that led to these genes.</p></td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="65%" cellspacing="0" cellpadding="2" border="1" bordercolor="#006699">
                <td align=center> <img src="images/unrooted.gif" width="165" height="113" alt="Image depicting a typical gene-based phylogenetic tree."></td>
            </table>
          </td>
        </tr>
		<tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td><p>The second panel, below, depicts three rooted trees that can be 
            drawn from the unrooted tree shown above, each representing the different 
            evolutionary pathways possible between these four genes. A rooted 
            tree is often referred to as an inferred tree. This is to emphasize that 
            this type of illustration depicts only the series of evolutionary 
            events that are inferred from the data under study and may not be 
            the same as the true tree or the tree that depicts the actual series 
            of evolutionary events that occurred.</p></td>
        </tr>
		<tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="85%" cellspacing="0" cellpadding="2" border="1" bordercolor="#006699">
                <td align=center> <img src="images/threetrees.gif" width="342" height="137" alt="Three examples of rooted trees which can be drawn from the unrooted tree described above."></td>
            </table>
          </td>
        </tr>
		<tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <p>To distinguish between the pathways, the phylogenetic analysis 
              must include at least one <b>outgroup</b>, a gene that is less 
              closely related to A, B, C, and D than these genes are to each other 
              (panel below). Outgroups enable the root of the tree to be located 
              and the correct evolutionary pathway to be identified. Let's say 
              that the four homologous genes used in the previous tree examples 
              come from human, chimpanzee, gorilla, and orangutan. In this case, 
              an outgroup could be a gene from another primate, such as baboon, 
              which is known to have branched away from the four species above 
              before the common ancestor of the species.</p>
          </td>
        </tr>
		<tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="50%" cellspacing="0" cellpadding="2" border="1" bordercolor="#006699">
                <td align=center> <img src="images/treeoutgroup.gif" width="192" height="180" alt="A diagram of a phylogenetic tree including an outgroup."></td>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4><i>Gene Trees Versus Species Trees&mdash;Why Are They Different?</i></h4>
            <p>It is assumed that a gene tree, because it is based on molecular 
              data, will be a more accurate and less ambiguous representation 
              of the species tree than that obtainable by morphological comparisons. 
              This may indeed be the case, but it does not mean that the gene 
              tree is the same as the species tree. For this to be true, the internal 
              nodes in both trees would have to be precisely equivalent, and they 
              are not. An internal node in a gene tree indicates the divergence 
              of an ancestral gene into two genes with different DNA sequences, 
              usually resulting from a mutation of one sort or another. An internal 
              node in a species tree represents what is called a <b>speciation 
              event</b>, whereby the population of the ancestral species splits 
              into two groups that are no longer able to interbreed. These two 
              events, mutation and speciation, do not always occur at the same 
              time.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" width="95%" cellspacing="0" cellpadding="5" border="2" bgcolor="#FFFFCC" bordercolor="#006699">
              <td align="center" bgcolor="#006699"><span class="title"><font color="#FFFFFF">Molecular 
                Phylogenetics Terminology</font></span></td>
              </tr>
              <tr> 
                <td> 
                  <p> 
                  <span class="inset"> 
                  <ul>
                    <li><b>Monophyletic</b>: two or more DNA sequences that are 
                      derived from a single common ancestral DNA sequence.</li>
                    <br>
                    
                    <li><b>Clade</b>: a group of monophyletic DNA sequences that 
                      make up all of the sequences included in the analysis that 
                      are descended from a particular common ancestral sequence.</li>
                    <br>
                    <li><b>Parsimony</b>: an approach that decides between different 
                      tree topologies by identifying the one that involves the 
                      shortest evolutionary pathway. This is the pathway that 
                      requires the smallest number of nucleotide changes to go 
                      from the ancestral sequence, at the root of the tree, to 
                      all of the present-day sequences that have been compared.</li>
                    <br>
                    <li><b>Molecular Clock Hypothesis</b>: states that nucleotide 
                      substitutions, or amino acid substitutions if proteins are 
                      being compared, occur at a constant rate, that is, the degree 
                      of difference between two sequences can be used to assign 
                      a date to the time at which their ancestral sequence diverged. 
                      The rate of molecular change differs among groups of organisms, 
					  among genes, and even among different parts of the same gene.  
					  Furthermore, molecular clocks require calibration with fossils 
					  to determine timing of origin of clades, and thus their accuracy 
					  is crucially dependent on the fossil record, or lack thereof, for 
					  the groups under study.  Fossil DNA older than about 
					  25,000&ndash;50,000 years is virtually empty of phylogenetic signal 
					  except in rare instances, and therefore traditional morphological studies 
					  of extinct and extant organisms remain a crucial component of 
					  phylogenetic analysis.</li>
                  </ul>
                  </span> </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>Systematics and NCBI</h3>
            <h4>The Taxonomy Project</h4>
            <p>The purpose of NCBI's Taxonomy Project is to build a consistent 
              phylogenetic taxonomy for the NCBI sequence databases. The <b><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">Taxonomy 
              Database</a></b> contains the names and lineages of every organism
			  represented by at least one nucleotide or protein sequence in the NCBI
			  genetic databases.  As of February 2003, this total is over 250,000 taxa.  
			  For current information, visit NCBI's 
			  <b><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/txstat.cgi">Taxonomy 
              Statistics Web page.</a></b>  The database is recognized 
              as the standard reference by the international sequence database 
              collaboration (GenBank, EMBL, DDJB, and Swiss-Prot).</p>
            <p>The <b><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">Taxonomy 
              Browser</a></b> is an NCBI-derived search tool that allows an individual 
              to search the Taxonomy database. Using the browser, information may be retrieved 
              on available nucleotide, protein, and structure records for a particular 
              species or higher taxon.  The Taxonomy Browser 
              can be used to view the taxonomic position or retrieve sequence 
              and structural data for a particular organism or group of organisms. 
              Searches may be made on the basis of whole, partial, or phonetically 
			  spelled organism names, and direct links to organisms commonly used in 
			  biological research are also provided. The Entrez Taxonomy system has the 
			  ability to display custom taxonomic trees representing user-defined subsets of 
			  the full NCBI taxonomy.</p>
            <p><b><a href="http://www.ncbi.nlm.nih.gov/sutils/taxik2.cgi">TaxPlot</a></b>, 
              another component of the Taxonomy project, is a research tool for 
              conducting three-way comparisons of different genomes. Comparisons 
              are based on the sequences of the proteins encoded in that organism's 
              genome. To use TaxPlot, one selects a reference genome to which 
              two other genomes will be compared. The TaxPlot tool then uses a 
              pre-computed BLAST result to plot a point for each protein predicted 
              to be included in the reference genome.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>BLAST: Detecting New Sequence Similarities</h4>
            <p>Currently, the characters most widely used for phylogenetic analysis 
              are DNA and protein sequences. DNA sequences may be compared directly, 
              or for those regions that code for a known protein, translated into 
              protein sequences. Creating phylogenies from nucleotide or amino 
              acid sequences first requires aligning the bases so that the differences 
              between the sequences being studied are easier to spot.</p>
            <p>The introduction of NCBI's <b><a href="http://www.ncbi.nlm.nih.gov/BLAST/">BLAST</a></b>, 
              or The <b>B</b>asic <b>L</b>ocal <b>A</b>lignment <b>S</b>earch 
              <b>T</b>ool, in 1990 made it easier to rapidly scan huge databases 
              for overt <b>homologies</b>, or sequence similarity, and to statistically 
              evaluate the resulting matches. BLAST works by comparing a user's 
              unknown sequence against the database of all known sequences to 
              determine likely matches. In a matter of seconds, the BLAST server 
              compares the user's sequence with up to a million known sequences 
              and determines the closest matches.</p>
            <p>Specialized BLASTs are also available for human, mouse, microbial, 
			  and many other genomes. A single BLAST search can compare a sequence of 
              interest to all other sequences stored in <b><a href="http://www.ncbi.nlm.nih.gov/Genbank/GenbankOverview.html">GenBank</a></b>, 
              NCBI's nucleotide sequence database. In this step, a researcher 
              has the option of limiting the search to a specific taxonomic group. 
              If the full scientific name or relationship of species of interest 
              is not known, the user can search for such details using NCBI's 
              <b><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">Taxonomy Browser</a></b>, 
              which provides direct links to some of the organisms commonly used 
              in molecular research projects, such as the zebrafish, fruit fly, 
              bakers yeast, nematode, and many more.</p>
            <p>BLAST next tallies the differences between sequences and assigns 
              a "score" based on sequence similarity. The scores assigned in a 
              BLAST search have a well-defined statistical interpretation, making 
              real sequence matches easier to distinguish from random background 
              hits. This is because BLAST uses a special algorithm, or mathematical 
              formula, that seeks local as opposed to global alignments and is 
              therefore able to detect relationships among sequences that share 
              only isolated regions of similarity. <b>Taxonomy-related BLAST</b> 
              results are presented in three formats based on the information 
              found in NCBI's Taxonomy database. The <b>Organism Report</b> sorts 
              BLAST comparisons, also called hits, by species such that all hits 
              to a given organism are grouped together. The <b>Lineage Report</b> 
              provides a view of the relationships between the organisms based 
              on NCBI's Taxonomy database. The <b>Taxonomy Report</b> provides 
              in-depth details on the relationship between all the organisms in 
              the BLAST hit list.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>COGs: Phylogenetic Classification of Proteins</h4>
            <p>The database of <b><a href="http://www.ncbi.nlm.nih.gov/COG/">Clusters 
              of Orthologous Groups</a></b> of proteins (COGs) represents an attempt 
              at the <b>phylogenetic classification</b> of proteins, a scheme 
              that indicates the evolutionary relationships between organisms, from 
              complete genomes. Each COG includes proteins that are thought to 
              be <b>orthologous</b>, or connected through vertical evolutionary 
              descent. COGs may be used to detect similarities and differences 
              between species, for identifying protein families and predicting 
              new protein functions, and to point to potential drug targets in 
              disease-causing species. The database is accompanied by the <b><a href="http://www.ncbi.nlm.nih.gov/COG/xognitor.html">COGnitor</a></b> 
              program, which assigns new proteins, typically from newly sequenced 
              genomes, to pre-existing COGs. A Web page containing additional 
              structural and functional information is now associated with each 
              COG. These hyperlinked information pages include: systematic classification 
              of the COG members under the different classification systems; indications 
              of which COG members (if any) have been characterized genetically and 
              biochemically; information on the domain architecture of the proteins 
              constituting the COG and the three-dimensional structure of the domains 
              if known or predictable; a succinct summary of the common structural 
              and functional features of the COG members, as well as peculiarities 
              of individual members; and key references.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp; </td>
        </tr>
        <tr> 
          <td> 
            <h4>HomoloGene</h4>
            <p><b><a href="http://www.ncbi.nlm.nih.gov/HomoloGene/">HomoloGene</a></b> 
              is a database of both curated and calculated orthologs and homologs 
              for the organisms represented in NCBI's <b><a href="http://www.ncbi.nlm.nih.gov/UniGene/index.html">UniGene</a></b> 
              database. Curated orthologs include gene pairs from the Mouse Genome 
              Database (MGD) at the Jackson Laboratory, the Zebrafish Information 
              (ZFIN) database at the University of Oregon, and from published reports. 
              Computed orthologs and homologs are identified from BLAST nucleotide 
              sequence comparisons between all UniGene clusters for each pair 
              of organisms. HomoloGene also contains a set of triplet clusters 
              in which orthologous clusters in two organisms are both orthologous 
              to the same cluster in a third organism. HomoloGene can be searched
			  via the Entrez retrieval system.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" border="0" cellspacing="0" cellpadding="5" width="95%" bgcolor="#FFFFFF">
              <tr> 
                <td> 
                  <table border="2" cellpadding="6" cellspacing="0" width="100%" bgcolor="#CCFFFF" bordercolor="#006699">
                    <tr> 
                      <td width="100%"><span class="inset"> <b><a href="http://www.ncbi.nlm.nih.gov/UniGene/index.html">UniGene</a></b> 
                        is a system for automatically partitioning GenBank sequences 
                        into a non-redundant set of gene-oriented clusters. Each 
                        UniGene cluster contains sequences that represent a unique 
                        gene, as well as related information, such as the tissue 
                        types in which the gene has been expressed and map location.</span><br>
                        </tr>
                  </table>
                </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>Entrez Genome</h4>
            <p>The whole genomes of over 1,200 organisms can be found in <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Genome">Entrez 
              Genomes</a>. The genomes represent both completely sequenced organisms 
              and those for which sequencing is in progress. All three main domains 
              of life&mdash;<b><a href="http://www.ncbi.nlm.nih.gov/genomes/static/eub.html">bacteria</a>, 
              <a href="http://www.ncbi.nlm.nih.gov/genomes/static/a.html">archaea</a></b>, 
              and <b><a href="http://www.ncbi.nlm.nih.gov/genomes/static/euk.html">eukaryotes</a></b>&mdash;
			  are represented, as well as many <b><a href="http://www.ncbi.nlm.nih.gov/genomes/static/vis.html">viruses</a>, 
              <a href="http://www.ncbi.nlm.nih.gov/genomes/static/vid.html">viroids</a>, 
              <a href="hhttp://www.ncbi.nlm.nih.gov/genomes/static/o.html">plasmids</a></b>, 
              and <b><a href="http://www.ncbi.nlm.nih.gov/genomes/static/euk_o.html">eukaryotic 
              organelles</a></b>. Data can be accessed hierarchically starting 
              from either an alphabetical listing or a phylogenetic tree for complete 
              genomes in each of six principle taxonomic groups. One can follow 
              the hierarchy to a variety of graphical overviews, including that 
              of the whole genome of a single organism, a single chromosome, or 
              even a single gene. At each level, one can access multiple views 
              of the data, pre-computed summaries, and links to analyses appropriate 
              for that level. In addition, any gene product (protein) that is 
              a member of a COG is linked to the COGs database. A summary of COG 
              functional groups is also presented in tabular and graphical formats 
              at the genome level.</p>
            <p>For complete microbial genomes, pre-computed BLAST neighbors for 
              protein sequences, including their taxonomic distribution and links 
              to 3D structures, are given in TaxTables and PDBTables, respectively. 
              Pairwise sequence alignments are presented graphically and linked 
              to NCBI's <b><a href="http://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml">Cn3D</a></b> 
              macro�molecular viewer that allows the interactive display of three-dimensional 
              structures and sequence alignments.</p>
          </td>
        </tr>
        <tr>
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h4>PDBeast: Taxonomy in MMDB</h4>
            <p>NCBI's Structure Group, in collaboration with NCBI taxonomists, 
              has undertaken <b>taxonomy annotation</b> for the three-dimensional 
              structure data stored in the <a href="http://www.ncbi.nlm.nih.gov/Structure/MMDB/mmdb.shtml">Molecular 
              Modeling Database</a> (MMDB). A semi-automated approach has been 
              implemented in which a human expert checks, corrects, and validates 
              automatic taxonomic assignments in MMDB. The <b><a href="http://www.ncbi.nlm.nih.gov/Structure/PDBEAST/pdbeast.shtml">PDBeast</a></b> 
              software tool was developed by NCBI for this purpose. It pulls text descriptions 
              of "Source Organisms" from either the original entries or user-specified 
              information and looks for matches in the NCBI Taxonomy database 
              to record taxonomy assignments.</p>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <table align="center" border="0" cellspacing="0" cellpadding="5" width="95%" bgcolor="#FFFFFF">
              <tr> 
                <td> 
                  <table border="2" cellpadding="6" cellspacing="0" width="100%" bgcolor="#CCFFFF" bordercolor="#006699">
                    <tr> 
                      <td width="100%"><span class="inset">The <b>Molecular Modeling Database (MMDB)</b> is a compilation 
                          of three-dimensional structures of biomolecules obtained 
                          from the <b>Protein Data Bank (PDB)</b>. The PDB, managed 
                          and maintained by the Research Collaboratory for Structural 
                          Bioinformatics, is a collection of all publicly available 
                          three-dimensional structures of proteins, nucleic acids, 
                          carbohydrates, and a variety of other complexes experimentally 
                          determined by X-ray crystallography and NMR. The difference 
                          between the two databases is that MMDB records reorganize 
                          and validate the information stored in the database 
                          in a way that enables cross-referencing between the 
                          chemistry and the three-dimensional structure of macromolecules. 
                          By integrating chemical, sequence, and structure information, 
                          MMDB is designed to serve as a resource for structure-based 
                          homology modeling and protein structure prediction.</span> 
					   </td>
                    </tr>
                  </table>
                </td>
              </tr>
            </table>
          </td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td> 
            <h3>The Importance of Molecular Phylogenetics</h3>
            <p>The field of molecular phylogenetics has grown, both in size and 
              in importance, since its inception in the early 1990s, attributable mostly 
              to advances in molecular biology and more rigorous methods for phylogenetic 
              tree building. The importance of phylogenetics has also been greatly 
              enhanced by the successful application of tree reconstruction, as 
              well as other phylogenetic techniques, to more diverse and perplexing 
              issues in biology. Today, a survey of the scientific literature 
              will show that molecular biology, genetics, evolution, development, 
              behavior, epidemiology, ecology, systematics, conservation biology, 
              and forensics are but a few examples of the many disparate fields 
              conceptually united by the methods and theories of molecular phylogenetics. 
              Phylogenies are used essentially the same way in all of these fields, 
              either by drawing inferences from the structure of the tree or from 
              the way the character states map onto the tree. Biologists can then 
              use these clues to build hypotheses and models of important events 
              in history. Broadly speaking, the relationships established by phylogenetic 
              trees often describe a species' evolutionary history and, hence, 
              its phylogeny&mdash;the historical relationships among lineages or 
              organisms or their parts, such as their genes. Phylogenies may be 
              thought of as a natural and meaningful way to order data, with an 
              enormous amount of evolutionary information contained within their 
              branches. Scientists working in these different areas can then use 
              these phylogenies to study and elucidate the biological processes 
              occurring at many levels of life's hierarchy.</p>
       </td>
        </tr>
		<tr> 
          <td>&nbsp;</td>
        </tr>
		 <tr> 
          <td><span><A HREF="#phylogenetics">Back to top</A></span></td>
        </tr>
                <tr> 
          <td>&nbsp;</td>
        </tr>
        <tr> 
          <td><span class="inset"> 
            <div align="left">Revised: April 1, 2004.</div>
            </span> 
        </tr>
      </table>
    </td>

      </table>
		<table align="bottom" width="600" border="0" cellspacing="0" cellpadding="5" bgcolor="#006699">
  <tr> 
  <td align="center" width="25%">&nbsp;
  </td> 
    <td align="center" width="25%"><a href="http://www.ncbi.nlm.nih.gov" class="sidebar"><span class="sidebar">NCBI</span></a></td>
    <td align="center" width="25%"><a href="http://www.nlm.nih.gov" class="sidebar"><span class="sidebar">NLM</span></a></td>
	    <td align="center" width="25%"><a href="http://www.nih.gov" class="sidebar"><span class="sidebar">NIH</span></a></td>
  </tr>
</table>

<br>
<table width="600" cellspacing="0" cellpadding="0" border="0">
<tr>
  <td align="center" width="25%">&nbsp;
  </td> 
  <td align="center" width="25%"><span class="nav"><a href="../privacy.html">Privacy Statement</a></span>
  </td>
  <td align="center" width="25%"><span class="nav"><a href="../disclaimer.html">Disclaimer</a></span>
  </td>
  <td align="center" width="25%"><span class="nav"><a href="../accessibility.html">Accessibility</a></span>
  </td> 
  </tr>
  </table>
 

</body>
</html>
