<base href="http://www.dartmouth.edu/~cbbc/courses/bio4/bio4-1997/16-HumanGenome.html">
<html>

	<head>
		<meta http-equiv="content-type" content="text/html;charset=iso-8859-1">
		<meta name="generator" content="GoLive CyberStudio">
		<title>A Brief History of the Human Genome Project</title>
	</head>

	<body background="back.GIF">
		<center>
			<h1>A Brief History of the Human Genome Project</h1>
			<h3>by George Cahill</h3>
		</center>
		<p>This is chapter 1 from: <font color="#000080">Gert, Bernard et al. 1996. &quot;Morality and the New Genetics: A Guide for Students and Health Care Providers.&quot;. Boston, MA: Jones and Bartlett, Publishers</font>. Used with permission of the author.</p>
		<p>
		<hr width="50%">
		</p>
		<center>
			<p><table width="100%">
				<tr>
					<th colspan="2">
						<h2>topics in these notes</h2>
					</th>
				</tr>
				<tr>
					<td><a href="#996758">Summary</a> </td>
					<td><a href="#996831">Let's Even Consider Sequencing!</a> </td>
				</tr>
				<tr>
					<td><a href="#996761">Antiquity</a> </td>
					<td><a href="#996837">Enter the Department of Energy</a> </td>
				</tr>
				<tr>
					<td><a href="#996765">Emerging Concepts</a> </td>
					<td><a href="#996847">DOE and NIH Take the Lead</a> </td>
				</tr>
				<tr>
					<td><a href="#996773">Mendel and Pea Counting</a> </td>
					<td><a href="#996851">Other Species</a> </td>
				</tr>
				<tr>
					<td><a href="#996781">Early Politics and Ethical Issues</a> </td>
					<td><a href="#996861">Mapping and Data Management</a> </td>
				</tr>
				<tr>
					<td><a href="#996789">Enter DNA</a> </td>
					<td><a href="#996869">National Coordination, Particularly in the Social Issues</a> </td>
				</tr>
				<tr>
					<td><a href="#996793">Mapping</a> </td>
					<td><a href="#996877">Finances</a> </td>
				</tr>
				<tr>
					<td><a href="#996785">Linkage</a> </td>
					<td><a href="#996900">Recent Happenings</a> </td>
				</tr>
				<tr>
					<td><a href="#996801">Clinical Genetics as a Medical Specialty</a> </td>
					<td><a href="#996894">The Plan</a> </td>
				</tr>
				<tr>
					<td><a href="#996811">Technologic Advances</a> </td>
					<td><a href="#996924">Some Problems</a> </td>
				</tr>
				<tr>
					<td><a href="#996821">Large Scale Mapping</a> </td>
					<td><a href="#996949">References</a> </td>
				</tr>
				<tr>
					<td><a href="#996817">DNA and Genetics Unite</a> </td>
					<td></td>
				</tr>
			</table></p>
		</center>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h2><a name="996758"></a>Summary</h2>
		</center>
		<p><font size="4">This chapter summarizes human genetics and its history with simple descriptions of modes of inheritance using the commonly-used terms from the genetic literature. It also describes current efforts to create genetic maps and to sequence the 3 billion bases in the human genome. An Appendix in the back of this volume summarizes the genetic principles of inheritance in humans.</font></p>
		<center>
			<h2><a name="996761"></a>Antiquity</h2>
		</center>
		<p>Awareness of the inheritance of both human appearance and behavioral characteristics dates from ancient times. In a similar vein, selective <strong>breeding</strong> of domestic animals and plants, both for greater yields and for ease of management, produced strains which maintained these better characteristics in subsequent generations. The evolution of today's corn from the minuscule wild maize of pre-Colombian America is one excellent example. Others include the draft horse, both beef and dairy cattle, and especially the various breeds of dogs for different uses such as herding, retrieving, pointing or simply companionship. These represent what can be done with intentional genetic manipulation employing appropriate selection for physical and behavioral traits.</p>
		<center>
			<h2><a name="996765"></a>Emerging Concepts</h2>
		</center>
		<p>Although many characteristics such as body height and build, hair and skin color and some behavioral traits were recognized in humans as familial, it was the inheritance of human abnormalities that led to a more definitive science. In India two millennia ago, Susrut described <strong>diabetes</strong> to be both &quot;passed through the seed&quot; (genetic) as well as influenced by environmental factors (&quot;an indolent spirit and love of rich foods and sweet wines&quot;). More scientific approaches began 200 years ago when the French physician Maupertuis recognized that <strong>polydactyly</strong> (more than 5 fingers) was passed from one generation to the next (McKusick, 1992). If one parent, either father or mother, expressed the abnormality, about 1/2 of the children of either sex would be similarly affected. Today we would call this mode of inheritance &quot;<strong>autosomal dominant</strong>&quot;.</p>
		<p>Similarly, <strong>hemophilia</strong> and <strong>color-blindness</strong> had both been noted to be familial but expressed only in males, and a number of papers in the early 19<sup>th</sup> century described the unique route of maternal inheritance with 1/2 of uncles on the mother's side expressing the disease. Likewise, the sisters of the affected boy might or might not pass the trait to their sons. Ancient Hebrew law even stated that circumcision could be obviated in a male infant if either his brother exhibited excessive bleeding or if the same tendency had been noted in a maternal uncle. Thus, &quot;<strong>sex-linked</strong>&quot; inheritance as we know it today was recognized in antiquity. Horner, a British physician, observed in 1876 that color blindness is inherited in boys from mothers whose brothers had the problem, another example of a classical sex-linked genetic history.</p>
		<p>Another British physician, John Adams, also in the early 19<sup>th</sup> century, made a more imaginative and incisive genetic observation when he realized that certain traits (or diseases) could come through the families of apparently normal parents (&quot;phenotypically&quot; normal but &quot;genotypically&quot; not normal, in genetic language). But these abnormal traits were manifested in only a small proportion of the children. Even more striking, this phenomenon occurred more frequently in kindreds in which the parents were not only normal but also were blood relatives (&quot;<strong>consanguinity</strong>&quot;). He had brilliantly described <strong>recessive inheritance</strong> in the early 1800's, a full century before the recognition of Mendel's great contributions to our understanding of inheritance (recognized independently by three European scientists some 30 years after Mendel's publication).</p>
		<center>
			<h2><a name="996773"></a>Mendel and Pea Counting</h2>
			<h3><a name="997985"></a><i>(also see <a href="01-Genetics.html#996783">Gregor Mendel</a>) </i></h3>
		</center>
		<p>Thus the genetic foundation for inherited disease in human beings had been laid without knowledge of genes, chromosomes, DNA and the like. It was the mathematician-monk <strong>Mendel</strong>, in his work on breeding garden peas, who put genetics into a quantitative discipline via the inheritance of &quot;factors&quot; which determine the physical features in subsequent generations such as plant height and flower color. Mendel correctly <i>postulated that two copies of each factor are present in each of the parents and only one copy of each factor in the sex products</i> - the &quot;gametes&quot;, or egg and sperm (pollen in plants) respectively. Mendel also pointed out the random or <i>independent inheritance of one pair of factors</i>, say pea or flower color, to another pair, &quot;wrinkled&quot; versus &quot;smooth&quot; and that <i>certain &quot;factors&quot; were dominant over the other</i>, a pea with one yellow factor and one green factor would be yellow in color (a yellow phenotype but a mixed genotype) and indistinguishable from a pea with both yellow factors (yellow phenotype and genotype). The latter, however, would produce only yellow peas in subsequent generations and the former, mixtures of the two colors. </p>
		<p>At the turn of the 20<sup>th</sup> century, Mendel's work, then four decades old, was discovered by several Europeans studying heredity and was publicized throughout the world's scientific community. Subsequently, Sutton, in 1904, attributed the small stained organelles in the nucleus or in the nuclear remnant of dividing cells, the &quot;<strong>chromosomes</strong>&quot;, as bearing the inherited traits. The British physician, Archibald Garrod characterized certain rare human diseases as being due to loss of a &quot;Mendel factor&quot;, presaging by decades the roles of genes in encoding proteins, especially enzymes in metabolic pathways. Several years later, in 1909, Johannsen called these inherited factors &quot;<strong>genes</strong>&quot;, providing the science of genetics with its name. </p>
		<p>Thus by 1910 genetics was off and running as a creditable and exciting science with Thomas Hunt Morgan using the common fruit fly, <i>Drosophila melanogaster</i>, as a model to study the inheritance of genetic traits. <i>Much of what we know of genetics today stems from these classical experiments of Morgan and his pupils</i>. In 1911 the X-chromosome was shown to be related to sexuality, 2 X's indicating female, and a single X with its counterpart the &quot;Y&quot;, being male in both humans and the fruit fly. E.B. Wilson attributed the sex-linked genes responsible for hemophilia and color-blindness in human beings to be located on the X-chromosome, similar to the many X-linked factors being described by the Morgan group in flies.</p>
		<center>
			<h2><a name="996781"></a>Early Politics and Ethical Issues</h2>
		</center>
		<p>The enthusiasm for the new science led to two movements, both leading to bad results, one for human well-being and the other for scientific progress. Since inheritance was deemed so important and was placed on a strong scientific basis by both Darwin and Mendel and their followers, the <strong>eugenic movement</strong> (probably better termed a cult) concluded that the future human population should be improved by selective breeding and culling. Kevles (1985) wrote an extensive treatise on this topic. This cult movement provides an excellent example of how the conviction that one is attempting to achieve a good objective, often leads one to adopt morally unacceptable means. The second unfortunate result was the attempt to quantify all human characteristics by the simple mathematical maneuvers of Mendel. Thus arose a computational school of genetic science with many of the world's great minds, Haldane, Huxley, Hogben, Galton, Jennings, attempting to calculate both population and individual characteristics, including physical and behavioral traits in terms of simple, single-gene models. This period of &quot;softness&quot; in genetic science even abetted the eugenicists in their naive attempt to improve the human gene pool. The final result of these activities was the <strong>Nazis</strong> attempt to select for racial purity with improved physical and emotional characteristics. This early history of eugenics shows how easy it is for a worthy goal to be perverted and how common it is for even great scientists to overestimate their scientific knowledge. <strong><i>We are now in another period of time when some scientists think that they know enough about genetics to attempt to achieve the same worthy goal of improving the future human population. It is one of the major goals of this book to provide such a clear account of morality that it will be extremely difficult for this goal to be used as a justification for using morally unacceptable means.</i></strong></p>
		<center>
			<h2><a name="996785"></a>Linkage</h2>
		</center>
		<p>Hogben did, however, suggest with his mathematical approach that genes situated close together on the same chromosome could be used for locating the anatomic position (locus) of other genes on that chromosome, in other words gene &quot;<strong>mapping</strong>&quot; by <strong>genetic linkage</strong>, and this was in the early 1930's. Geneticists studying mice, fruit flies and the bread mold, Neurospora, were cataloguing numerous traits (&quot;factors&quot; or, more properly, genes) and found that variations in a single gene (an &quot;<strong>allele</strong>&quot;) would frequently be inherited with a specific variation in another gene (a particular allele of this second gene). Thus these two genes must be close together or &quot;linked&quot;. Mice were found to have 20 large &quot;<strong>linkage&quot; groups</strong>, matching the number of chromosomes in mice. Soon thereafter, Haldane's group calculated how close together in genetic terms, were hemophilia and color-blindness on the human X-chromosome, since over many generations the two would occasionally separate due to crossing-over (recombination) in gamete (eggs or sperm) production. Thus the concepts of linkage and its mathematical measurements were firmly established.</p>
		<center>
			<h2><a name="996789"></a>Enter DNA</h2>
		</center>
		<p>Keep in mind that all of the above was &quot;paper&quot; biology centered about breeding experiments in humans, animal models and plants. Little was known about the underlying structure and function of the genetic information. The author of this chapter was taught (in 1944) in college biology that genes were located in the protein-nucleic acid complexes in the &quot;48&quot; (!) chromosomes in human beings with the genetic specificity for the code probably residing in the protein part since it contained 20 different building blocks (the amino acids) compared to the paltry four building blocks in nucleic acids (A,G,C and T). The latter supposedly provided some kind of order or structural support! At this same time, Avery and colleagues at the Rockefeller Institute were finding that one strain of pneumococcus could be made into another strain simply by transferring the nucleic acid component (DNA) free of all protein [see <a href="03-DNA&Chromosomes.html">Avery, MacLeod, McCarthy (1944)</a>]. It still took a number of years, however, before this most fundamental contribution, that DNA is the vehicle for genetic information, was entered into the standard biological texts.</p>
		<center>
			<h2><a name="996793"></a>Mapping</h2>
		</center>
		<p>Mapping, or more properly, <i>assignment of human genes to locations on various chromosomes</i> proceeded very slowly, since controlled breeding experiments in humans could not and cannot be done. Mohr noted in 1951 that a gene for a blood protein factor, Lutheran, in some families appeared to be associated with another gene controlling salivary secretion of a red blood cell protein. In other families it was not associated with the secretion gene. Thus &quot;secretor&quot; and Lutheran appeared linked and presumably close together on the same chromosome. Both of these are now assigned to Chromosome 19, physically corroborating the earlier family linkage studies of Mohr.</p>
		<p>The major contribution of Watson and Crick came in 1953, the description of the molecular structure of the genetic material, the double helix. More important was the emphasis on the specific pairing of the two sets of nucleic acids, <code><font size="4">A</font></code>to <code><font size="4">T</font></code>and <code><font size="4">G</font></code>to <code><font size="4">C</font></code>. This pairing is the basis of all nature since it is not only central to the accurate duplication of genes in cell growth and reproduction, but also in how genes are made into specific messages for synthesis of specific proteins.</p>
		<p>This pairing of the four bases in DNA, A to T and G to C, is the basis of today's molecular science. A and T bind and fit like a key in a lock and the same for G and C. A chromosome is simply an immense single molecule of alternating sugar and phosphate molecules with one of the four bases attached to each molecule of sugar (see <a href="03-DNA&Chromosomes.html">DNA Structure</a>). Thus if the sequence <code><font size="4">-T-A-G-G-C-T-A-G-G-C-</font></code> is located somewhere in a given chromosome, its complementary strand in the double helix would contain <code><font size="4">-A-T-C-C-G-A-T-C-C-G-</font></code>. If a chemist were to make the latter sequence and label it by adding a colored tag or a radioactive element (creating a &quot;probe&quot;) and add it to a test tube full of DNA, the labeled sequence would search out and bind to its complement wherever it appeared in the millions to billions of DNA sequences in the test tube. The labeled probe could even be added to a microscopic slide containing the DNA to be examined and anatomically identified as to where it binds, for example to which chromosome and even to where on that chromosome. By random chance alone, these 10 bases would be expected to find their complementary counterpart once in every 4<sup>10</sup> stretches of 10 bases or once per 1,048,576 bases. This concept is fundamental to the locating (&quot;mapping&quot;) of genes and non-gene sequences throughout the genome.</p>
		<p>&middot;&middot;&middot;&middot;</p>
		<center>
			<h2><a name="996801"></a>Clinical Genetics as a Medical Specialty</h2>
		</center>
		<p>Human genetics was, however, by today's standard, proceeding at a snail's pace. Tijo and Levan in 1956 found human beings had 46 and not 48 chromosomes, and several years later, Lejeune noted a 47th chromosome in individuals with Down syndrome. Other significant genetic concepts were formulated at about that time; Allison, in England, proposed the provocative suggestion of a possible protection induced by the sickle cell trait against malaria. A trait, thereby, can be selected for in a certain environment where there is sufficient pressure for its positive effects over its deleterious effects.</p>
		<p>Another notable contribution was made by Barr who found female cells bear a blob of chromatin in their nucleus. It was later shown by the British mouse geneticist, Mary Lyon, that females inactivated early in embryonic life one of their two X chromosomes, the inactive one persisting as the &quot;Barr&quot; body. This was used for sex identification of athletes until recently. There were errors in the test which diminished its usefulness. Today's testing is done with molecular probes for sequences specific to the Y chromosome. Thus cells in both males and females have but one functioning X chromosome. In the female, however, the inactivation is random and therefore the female carrier is usually phenotypically normal with half her cells bearing enough genetic material to do the job.</p>
		<p>By the 1960's genetics had become a medical subspecialty and in 1966 Victor McKusick at Johns Hopkins Medical School published the first edition of &quot;Mendelian Inheritance in Man&quot; containing 1500 inherited characteristics, or in proper terms, &quot;mendelizing phenotypes&quot;. Again, the &quot;phenotype&quot; is the physical expression of a gene whereas the &quot;genotype&quot; refers to the genetic material of that gene. Of these 574 had been clearly identified. In the 11th edition of 1994, now two volumes, there are 6,678 entries: 4,458 autosomal dominant, 1,730 recessive, 412 X-linked, 19 Y-linked, and 59 located in mitochondria (McKusick, 1994).</p>
		<p>Although linkages had been found, especially on the X chromosome as exemplified by sex-linked inheritance in hemophilia and color-blindness, it wasn't until 1968 that an autosomal (meaning not sex-linked) assignment of linkage was made by Donahue, an associate of McKusick. He observed a peculiar microscopically-visible stretch of chromatin on his own largest chromosome (Chromosome #1). Looking at a number of blood factors Donahue found that in both himself and his relatives, an allele of the Duffy blood factor was linked to this observable physical change in their 1st chromosome. Thus &quot;Duffy&quot; was assigned to Chromosome #1.</p>
		<center>
			<h2><a name="996811"></a>Technologic Advances</h2>
		</center>
		<p>A major contribution to mapping and linkage of genes was subsequently made by Weiss and Green who joined hamster and human cells into hybrid cells in tissue culture. In subsequent generations the human chromosomes were lost, leaving a hamster cell line with, at times, a single human chromosome. When specific markers for human genes such as molecular probes for sequences in that gene or for their protein products were used, the nearby genes on that remaining human chromosome could be identified and assigned to that chromosome. Thus linkage was more easily established. This kind of mapping is termed &quot;physical mapping&quot; since it entails chemical technology to physically locate a stretch of DNA. The other kind of mapping, genetic or &quot;Mendelian&quot;?mapping, requires large scale breeding experiments, or, in humans, large pedigrees to calculate by linkage with other known genes or identified stretches of DNA where the unknown factor is located.</p>
		<p>Another major contribution was made by Caspersson who designed a greatly improved method for staining chromosomes producing bands of differing intensity depending on the relative densities of G-C versus A-T in that region. Thus not only could gross abnormalities be more easily recognized, but the experienced eye could identify each human chromosome and the specific regions on that chromosome as recognized by their banding appearance. Nevertheless by the mid 1970's only several hundred genes in human beings had been mapped, significant progress, but slowly, realizing that the human genome contains perhaps 100,000 genes!</p>
		<center>
			<h2><a name="996817"></a>DNA and Genetics Unite</h2>
		</center>
		<p>At the first human gene mapping workshop held in New Haven in 1973, new data on gene mapping were sparse as exemplified by no genes yet assigned to chromosomes 3, 8, or 9. Medical usage of the data was also infrequent. Nevertheless Ruddle at Yale, with shrewd foresight, began his computer compilation of the information in anticipation of the oncoming plethora of data as the science matured. Kan and colleagues in San Francisco used molecular hybridization (a fancy term for A-T and G-C binding between a stretch of DNA and its complementary counterpart as a probe) to identify losses of globin genes in a relatively common type of anemia in human beings, thalassemia (Mediterranean or Cooley's) anemia. Two years later, in a very significant observation, Kan and Dozy (1978) noted that near the sickle cell gene was a stretch of DNA which varied between most blacks and which could be used as a marker in a given family for linkage with the sickle cell gene. This variant (properly termed a &quot;polymorphism&quot;) served as the first use of these common molecular differences between one person and another as a method for both diagnosis (by being linked to a disease gene) and subsequently for general mapping over the entire genome. The scientific community was than prepared for a quantum leap in molecular genetics and the human was becoming the principal model.</p>
		<center>
			<h2><a name="996821"></a>Large Scale Mapping</h2>
		</center>
		<p>Several events occurred in 1979, a publishing of the linkage diagnosis of sickle cell anemia described above, and, in similar studies in England, Solomon and Bodmer suggested that the same technology could be applied to other genetic diseases and traits. In other words, unique variants in the genome sequence could be used as markers for nearby abnormal genes. At a small, rather informal conference held in 1978 in Alta, Utah, several researchers led by Botstein suggested the use of these variants spread out over the human genome could predict in given kindreds the location of many genes by their genetic linkage to the variant. Botstein, Skolnick, Davis and White published this suggestion in 1980 and it is now a milestone in genetics. White, in the fall of 1979, with his research fellow, Wyman, at the University of Massachusetts Medical School, isolated their first useful anonymous probe, meaning a variable stretch of DNA with no known function. This was mapped several years later to Chromosome #14. This probe showed many variations between individuals (highly &quot;polymorphic&quot;?and therefore very informative in genetic lingo). Subsequently it and many others could be used as road-markers in the overall effort to find many milestones up and down all of the 23 human chromosomes. White subsequently moved to Utah for better access to the large Mormon kindreds assembled by Skolnick in which the four grandparents, both parents and many children (6 or more, and in a few families, over a dozen) could be used for linkage analysis of polymorphisms and disease-related genes. Thus was initiated the detailed mapping of the human genome.</p>
		<p>This started the ball rolling for a number of laboratories to search for variations&macr;called RFLP's (restriction fragment length polymorphisms) and their linkage to other known loci and to disease entities. Within the next few years about 1/3rd of the useful and available probes were generated by White's large lab in Salt Lake City, about 1/3rd by a commercial concern, Collaborative Research in Boston directed by Helen Donis-Keller and her team and the remainder by a number of laboratories about the rest of the world.</p>
		<p>Nobel laureate Jean Dausset assembled a team of physicians and molecular biologists in Paris (&quot;CEPH&quot; for Centre Etude Polymorphisme Humaine) which prepared samples of DNA from originally 40 very large families with 3 living generations, including 27 from White's Utah Mormon collection, the immense Venezuela pedigree collected by Nancy Wexler for her studies on Huntington's disease, 10 French families and 2 others from the Pennsylvania Amish genetic studies by the Johns Hopkins Group led by McKusick. These DNA pools from 600 individuals have been made available by CEPH for investigators about the world mapping genes for diseases and other genetic loci of their interest, sparing the need for them to assemble similarly large &quot;control&quot; kindreds for linkage analysis.</p>
		<p>Over the next few years the locus of Huntington's disease (Chrom. 4, Gusella et al, 1983), adult polycystic renal disease (Chrom. 16, Reeders et al, 1985), muscular dystrophy ( Chrom. X, Kunkel et al, 1987) and cystic fibrosis (Chrom. 7, by a large collaborative effort between Collins et al and Tsui et al and others, 1989) were all mapped using this technology. Subsequently the product of the isolated defective genes for muscular dystrophy and for cystic fibrosis were characterized by &quot;reverse genetics&quot;. This means finding a stretch of DNA known to bear the disease by linkage analysis, then finding the gene in the stretch and finally, determining what the gene does, where it is physically expressed, in what tissue and where in the cell. The gene for Huntington's disease has just been found by this process and it is hoped that the precise role of this gene in causing the pathology can now be characterized, hopefully, using that knowledge for an eventual prevention or cure. Mapping genes, normal and diseased, was off and running.</p>
		<center>
			<h2><a name="996831"></a>Let's Even Consider Sequencing!</h2>
		</center>
		<p>In May 1985 a meeting was convened in Santa Cruz, California, by Robert Sinsheimer, a distinguished molecular biologist and also a very senior administrator at the University of California. A number of biologists active in genetics and gene mapping were present, including Nobelist Walter Gilbert. The dramatic proposition was made that the entire human genome should not only be mapped with scattered but specific road-markers along the highway, but also sequenced, meaning determining the precise order of each A, G, C and T. A single copy (the normal has 2 copies each of 23 chromosomes, one copy from each parent) contains approximately 3 billion bases (A, G, C or T). At the time, analysis of a single base cost over $10 and it took a good scientist a day to sequence 50 to 100 bases. Thus the suggestion to sequence the genome was not only imaginative and ambitious, but prohibitively expensive.</p>
		<p>It should be mentioned that by 1991, the technology had improved sufficiently to allow up to 10,000 bases to be determined in a single day at about a dollar a base (Hunkapiller et al, 1991). In 1993, sequencing was being done by several laboratories using robotic analyzers at 500,000 bases daily, costing 10-15 cents/base. Mapping, using primarily the RFLP technology by 1985, as summarized in the publication of Ruddle's computerized resource in New Haven, HGML, (Human Gene Mapping Library) listed over 1,200 locations for both genes and anonymous probes. By the 10th International conference held in Paris in 1989, there were listed 5,510 locations, 8,450 probes including data on just under 2,000 RFLP's. The science was moving rapidly. But sequencing and mapping are greatly differing tasks in size, expense and complexity, as the following will attest. It should be stated that very little attention was paid at that time to any moral or ethical issues which might ensue. The potential benefits for cancer, genetic diagnosis in pregnancy and the genetic prediction for diseases in mid and later life and even the proper prevention of genetic problems by gene manipulations were all so attractive that considerations of the privacy of the data, their possible prejudicial use in insurance, employment and many other problematic applications were essentially obscured by the enthusiasm of the protagonists (Tauber and Sarkar, 1992; Muller-Hill, 1993).</p>
		<center>
			<h2><a name="996837"></a>Enter the Department of Energy</h2>
		</center>
		<p>The Department of Energy had a number of laboratories (residues of the atomic bomb project and subsequently nuclear energy for power production) dealing with the biological effects of irradiation, located at Oak Ridge, Los Alamos, Livermore and Berkeley. Using their expertise in engineering technology derived from their massive projects in atomic energy and its potential dangers, they had already made several major contributions toward the genome project. One of these was the capacity to physically separate chromosomes by their size and staining, thus superseding the complex biological separation using human-hamster cell hybrids previously developed for linkage and mapping. Linkage was therefore greatly simplified since instead of dealing with all 23 pairs of chromosomes, one had either single chromosome collections or pools of several chromosomes in separated samples. The field moved forward at a greatly accelerated pace (McKusick, 1989a).</p>
		<p>There were also politics. The studies of Hiroshima survivors fortunately failed to show a major genetic impact of atomic irradiation, although there were significant increases in malignancy rates in the survivors (Lindee, 1995). What therefore should DOE do with all their assembled biologists and physicists in the national labs? One major contribution had already been started, namely, Genbank, a computerized repository for the primary DNA sequences of genetic material from all organisms reported in the literature. This was located at Los Alamos with a collaborating counterpart at the European Molecular Biology Laboratories (EMBL) in Heidelberg, Germany. By 1991, some 60 million bases were recorded, about 1/2 being human and the remainder from bacteria, mice, fruit flies, etc. This number doubled over the subsequent two years to well over 100 million bases.</p>
		<p>A biophysicist and administrator in DOE, Charles DeLisi, convened a meeting in Santa Fe in early 1986 to spell out the role of the Department in sequencing the entire human genome, following up on the previous meeting at Santa Cruz. A similar meeting was subsequently convened a month later by James Watson at Cold Spring Harbor. Already there was rising opposition from a number of quarters which ranged from calling the task a political boondoggle, a task which would drain funds from other more needy science or a task which would be scientifically not very productive since a major part of the genome is not made up of genes but &quot;filler&quot; DNA or, more simply, &quot;junk&quot;. Finally, there also began a concern with the moral problems that might result from learning too much about an individual's genome and particularly with the eventual use of that information. More will be said on this subsequently; in fact, it is the purpose of this volume! Also animosity arose between DOE and other federal agencies. Scientists, both federal and in the private sector, aligned either strongly for or strongly against the project. The Nobel Laureate, Renato Delbucco published a strongly protagonistic article in Science, stating that knowledge of cancer, perhaps even its potential cure, would result from genome sequence information. Other leading scientists (Davis, Leder, Rechstein) took the opposing side and publicized their objections accordingly. This debate between prominent scientists added even more concern to those who were raising the ethical and moral issues, both for and against the genome project.</p>
		<p>The Howard Hughes Medical Institute, whose asset, Hughes Aircraft, had just been sold to General Motors for over 5 billion dollars, had been supporting not only the Utah operation of Ray White and several other mappers, but also Ruddle's data base, The Human Gene Mapping Library (HGML). The Institute, in an ecumenical attempt, convened authorities from both DOE and the NIH and invited a number of foreign representatives. The meeting, held in Bethesda, was chaired by Sir Walter Bodmer of England, a major contributor to knowledge of human genetics and immunology. A number of advances were presented, including advances in sequencing technology, techniques for isolating large pieces of DNA using pulsed field gels, mapping techniques used to connect contiguous pieces of DNA, and many others. The meeting did get the attention of the public, and, more importantly, of a number of government agencies and both the legislative and executive branches. The genome was now launched into the political sphere and into the public and scientific media. Robert Cook-Deegan has summarized these events in a volume covering the history of the genome between 1986 and 1990 (Cook-Deegan, 1993).</p>
		<center>
			<h2><a name="996847"></a>DOE and NIH Take the Lead</h2>
		</center>
		<p>Over the next year a number of other meetings were held by DOE and NIH as well as independent hearings by the Congressional Office of Technology Assessment (OTA) and by the National Academy of Science. These activities and their published reports all culminated in 1988 in the establishment of the Genome Office at the National Institute of Health directed by Nobelist James Watson. This subsequently became the National Center for Human Genome Research (October 1989) by Congressional authorization. Advisory boards were created to serve both agencies (NIH and DOE). Meetings have been held twice annually on succeeding days since several committees are &quot;joint&quot;, especially the one on data (Joint Informatics Task Force). Also initiated was the NIH-DOE ELSI (Ethical, Legal and Social Issues) Working Group. On the recommendation of James Watson 3% of the total funds were to be devoted to these activities, a rather bold and innovative move both by Watson himself and by the NIH. This volume is supported by a 3-year grant awarded by the National Center and the first to have a philosopher as Principal Investigator.</p>
		<center>
			<h2><a name="996851"></a>Other Species</h2>
		</center>
		<p>Although the genome task is labeled &quot;Human&quot;, other species are included for a number of reasons. Bacteria, particularly E. coli, have served as the original models for molecular biology, such as how genes are turned on or turned off, thus the study of bacterial genetics is directly relevant to human disease. By 1995, the entire 1,830,137 base pairs of the bacterium, Haemophilus influenzae RD, had been sequenced and its genes mapped by a large team headed by J. Craig Venter at the Institute for Genomic Research (TIGR) in Gaithersburg, MD (Venter, 1995). A tiny worm, C. elegans, has less than 1000 cells in its total body and has served as a superb model for how an animal's genome programs itself from a single fertilized cell into a multi-celled adult. Its generation time is only 3.5 days and it has but 100 million base pairs in its genome compared to the 3 billion in human beings. The fruit fly is also included; it is the historic model for genetics thanks to its ease in breeding and the very large amount of knowledge already accumulated starting with Morgan's studies 8 decades ago. Other relevant forms such as yeast and subhuman primates are included, but most important is the mouse (vide infra).</p>
		<p>Genes are evolutionarily conserved. If a gene works well in bacteria and plants, it is probably similar in both structure and sequence in animals. Thus certain genes in mice and humans are almost chemically identical, and probes used to identify the gene in one oftime react with the homologous gene in the other. Of interest, however, is that the order of the genes on the chromosomes is also usually conserved. Thus if there is linkage between two genes in the mouse, the linkage between the two is probably also in human beings. Luckily, it is morally acceptable to breed mice, for mice are easier to breed and study compared to humans, mapping is far easier in mice, and these data can then be extrapolated back to humans. For example the 17th chromosome in mice is homologous in large part to the 11th in human beings and of the 35 mapped loci in both organisms on these chromosomes, all but two are ordered into the same sequence. Another major reason for interest in the mouse has been the recently-developed technology, primarily by Capecchi of the University of Utah and Smithies, now at the University of North Carolina, which creates mutations or deletions of specific genes in the mouse and therefore produces genetic models for disease. Thus knowledge of mouse genes, of their locations and of their sequences is directly relevant to the human genome project. Another major reason for the interest in the mouse is that many diseases are polygenic and result from the interaction of a number of inherited factors. Hypertension, coronary artery disease, juvenile and most maturity-onset diabetes and most other of the common human diseases are polygenic. Sorting out the relative roles of these multiple genes necessitates both breeding and molecular genetics, and obviously this can only be done in the mouse.</p>
		<p>Finally, there is also now a great flurry to map and eventually to sequence genes in plants and in domestic animals for both agricultural and pharmaceutical purposes. The cow genome has just been partially mapped with probes for each of its 30 pairs of chromosomes. New strains of vegetables resistant to pathogens as well as to frost are being developed. Human proteins are now being produced by human genes functioning in domestic animals for subsequent harvest and clinical usage without fear of human infecting viruses such as AIDS.</p>
		<p>A few words must be said about baker's yeast (Saccharomyces cerevisiae, to be more proper). Maynard Olson and colleagues in St. Louis have been successful amongst others in inserting fragments of human genes into yeast. The yeast are then grown and examined for which sets of genes are together in a given fragment. This technology has greatly simplified mapping of genes to specific locations on specific chromosomes.</p>
		<center>
			<h2><a name="996861"></a>Mapping and Data Management</h2>
		</center>
		<p>With all the aforementioned biological and political history we should examine the present status of the genome project in the United States and other nations where significant programs are in operation. Once the movement for mapping the human genome began to accelerate in the late 70's and early 80's there arose a number of needs for coordination, communication and collaboration at all levels. A map with markers at approximately 10 centimorgans would necessitate approximately 300-400 evenly spaced probes with each having a high level of polymorphism so that they could be used for linkage studies in pedigrees. A centimorgan (cM) approximates 1 million bases, although in the mendelian map derived from genetic linkage studies (See appendix, Figures 3 and 4) this is highly variable and where there is a high degree of recombination may be as few as 10-20,000 bases, and, where recombination is infrequent, perhaps several million bases. More problematic is that at any given location, recombination may be more or less frequent between oogenesis and spermatogenesis, meaning that the linkage map may be larger or smaller than the anatomic (physical) map at any location between male and female. There are really, therefore, two genetic maps, one for males and one for females. More simply stated, since the &quot;genetic&quot; distance between two genes is measured in centimorgans (% frequency of recombination between two points), the probability that they will separate in the formation of an egg or sperm, say, if 10 centimorgans apart is 10 in 100. In egg formation this may be 20 million bases between two genes whereas in sperm formation, perhaps 5 million. Thus the genetic map at this location would be 4 times larger for oogenesis whereas the physical map is the same for both processes. In other areas, the male distance may be larger. The mechanisms for this difference remain obscure.</p>
		<p>Several data bases for biologic information were already in operation at this time, including the Protein Data Base for molecular structure at Brookhaven National Laboratories, the Protein Information Resource started by Dr. Margaret Dayhoff located at Georgetown University for the storage of amino acid sequences in proteins, and Genbank for the storage of DNA sequences, located in Los Alamos at the National Laboratories of the Department of Energy. Genbank is linked administratively, scientifically and literally by computer to a similar facility, EMBL, in Heidelberg, Germany. As mentioned previously, the Human Gene Mapping Library was started by Frank Ruddle in New Haven for the storing of human genetic information in a number of data bases accessible to scientists via computer over the telephone. The Jackson Laboratory at Bar Harbor, ME, had a similar data bank for mouse information and it was in close contact with a parallel data base outside London under the direction of Dr. Mary Lyon.</p>
		<p>The term &quot;informatics&quot; had been coined by Dr. Donald Lindberg, Director of the National Library of Medicine, and it was clear that informatics would play a central role in the genome effort with the accession, organizing, banking, networking and distribution of the immense amount of information accumulating as the genome effort progressed. Clusters of small data bases have evolved as well as an exponential growth of those mentioned above. The principal resources, however, continue to be the Los Alamos data pool of DNA sequences and the Gene Data Base (GDB) for mapping information (formerly Ruddle's Human Gene Mapping Library &quot;HGML&quot;). It is now located at the Johns Hopkins Medical School.</p>
		<center>
			<h2><a name="996869"></a>National Coordination, Particularly in the Social Issues</h2>
		</center>
		<p>One of the initial and most important roles of the National Center for Human Genome Research, led by Nobel Laureate James Watson and its corresponding directorate in the Department of Energy, led by Dr. David Galas, was to develop guidelines for data accession and storage as well as intercommunication between data bases. For these the Joint Information Task Force was set up under Dr. Dieter Soll of Yale and Dr. Mark Pearson of the Dupont Company. They have assembled a detailed report containing chapters dealing with computer technology, analysis of genomic data, collection and management of laboratory data, communication between data bases and training and development in the informatics of genomics. </p>
		<p>Another joint committee, the ELSI (Ethical, Legal and Social Issues), entitled a &quot;Working Group&quot; and chaired by Dr. Nancy Wexler, was formed. She was a logical person to head up this group which is so central to the humane aspects of the overall effort, being a well-trained social scientist from a family with a history of Huntington Disease. This genetic abnormality affects brain function in mid-life and is inherited in a dominant fashion (See appendix, Figures 4 &amp; 5) meaning one half of the children are at risk. Thus the molecular diagnosis of those at risk versus those not at risk, which can now be done with much certainty, poses a major problem in medical and emotional management, including possible amniocentesis and the pros and cons of abortion if the child is diagnosed as a probable carrier of the disease gene. (Chapt. 9). Dr. Wexler made major contributions to the efforts to make a molecular diagnosis by assembling an extremely large kindred in Venezuela which was used for the linkage analysis to map the disease gene.</p>
		<p>The ELSI group considers as its major priority, issues relating to the validity of and access to genetic tests and related information, the fairness in the use of the information in employment and insurance, the storage, confidentiality and accessibility of genetic information and, lastly, information and education prepared for the public to deal with the ethical, legal and social problems created by the availability of the genetic information.</p>
		<center>
			<h2><a name="996877"></a>Finances</h2>
		</center>
		<p>The growth of the American genome initiative is reflected in the federal funds added to the governmental support of basic biomedical science which approximates several billion dollars annually:</p>
		<center>
			<p><table border="3">
				<tr>
					<td colspan="9">
						<center>
							<strong><b>Genome Expenditures ($ millions/annum) </b></strong></center>
					</td>
				</tr>
				<tr>
					<td>
						<center>
							<b>Fiscal Year: </b></center>
					</td>
					<td>
						<center>
							<b>'87 </b></center>
					</td>
					<td>
						<center>
							<b>'88 </b></center>
					</td>
					<td>
						<center>
							<b>'89 </b></center>
					</td>
					<td>
						<center>
							<b>'90 </b></center>
					</td>
					<td>
						<center>
							<b>'91 </b></center>
					</td>
					<td>
						<center>
							<b>'92 </b></center>
					</td>
					<td>
						<center>
							<b>'93 </b></center>
					</td>
					<td>
						<center>
							<b>'94 </b></center>
					</td>
				</tr>
				<tr>
					<td>National Inst. Health </td>
					<td>
						<center>
							0 </center>
					</td>
					<td>
						<center>
							17 </center>
					</td>
					<td>
						<center>
							28 </center>
					</td>
					<td>
						<center>
							60 </center>
					</td>
					<td>
						<center>
							88 </center>
					</td>
					<td>
						<center>
							105 </center>
					</td>
					<td>
						<center>
							106 </center>
					</td>
					<td>
						<center>
							129 </center>
					</td>
				</tr>
				<tr>
					<td>Dept. of Energy </td>
					<td>
						<center>
							5 </center>
					</td>
					<td>
						<center>
							12 </center>
					</td>
					<td>
						<center>
							18 </center>
					</td>
					<td>
						<center>
							28 </center>
					</td>
					<td>
						<center>
							47 </center>
					</td>
					<td>
						<center>
							61 </center>
					</td>
					<td>
						<center>
							65 </center>
					</td>
					<td>
						<center>
							? </center>
					</td>
				</tr>
				<tr>
					<td>Agriculture </td>
					<td>
						<center>
							0 </center>
					</td>
					<td>
						<center>
							0 </center>
					</td>
					<td>
						<center>
							0 </center>
					</td>
					<td>
						<center>
							0 </center>
					</td>
					<td>
						<center>
							15 </center>
					</td>
					<td>
						<center>
							15 </center>
					</td>
					<td>
						<center>
							15 </center>
					</td>
					<td>
						<center>
							? </center>
					</td>
				</tr>
			</table></p>
		</center>
		<p>Of the 1991 NIH funds of 88 million dollars, 28 were devoted to human mapping and sequencing. Of the remainder, 8 to the mouse, 3 to the bacterium E. coli, 2 to the worm, C. elegans, 1 to the fruit fly, Drosophila, 4 million to all other microorganisms and 1/2 million to plants and yeast. These amounts represent only the added funds earmarked by congress for the new effort and do not include the approximate 1 to 2 billion dollars of the total 1991 NIH budget of 8 billion (10 billion, 1993) which involve biomedical science related to genetics. One might even say that the entire NIH and Energy health science budgets deal with genetics since genes and their variations (alleles) underlie all life, healthy or diseased.</p>
		<p>On an international scale, the Human Gene Organization (HUGO), a body of geneticists and molecular biologists elected in an academy fashion by nomination and subsequent voting by the membership, was formed and incorporated in Switzerland in September 1988 (McKusick, 1989b). HUGO's role is to coordinate communication and data exchange across international boundaries. HUGO currently hosts the single chromosome mapping workshops and has also assumed responsibility for the biannual international Human Gene Mapping conferences, the last being held in Baltimore in September, 1992 with Sir Walter Bodmer, President of HUGO, serving as its Chairman. A number of other organizations have arisen, including national groups in the United Kingdom, France, the Commonwealth of Russian Republics (formerly the USSR), Japan and others. So far the U.S. has made the major contributions in funding and information produced, probably well over 3/4ths of the total effort. However, in the consideration of the moral and ethical issues, some of the first meetings were held in Europe, particularly, (and with good historical reasons) in Germany. Thomas Caskey then of Houston, a molecular-oriented human geneticist replaced Bodmer as President of HUGO in 1993.</p>
		<center>
			<h2><a name="996894"></a>The Plan</h2>
		</center>
		<p>The Department of Energy and the National Institutes of Health jointly launched a five-year plan to start in fiscal 1991. In essence, this was to be the first leg of a more ambitious series of 3 five-year plans with the ultimate objective that the entire 3 billion bases of a single copy of the human genome would be mapped and sequenced. The first five year proposal, as spelled out in the formal document, was to saturate the genome with markers spaced originally 10 cM apart, meaning about 300 to 400 markers, and, subsequently, at 1cM, necessitating 10 times that number. Thus the entire genome would be &quot;mapped&quot; at equally-spaced and uniquely specific loci. Sequencing, on the other hand, would focus on areas of interest mainly directed to human disease. Currently in Genbank, the Los Alamos gene data bank operated by the Department of Energy, there are stored about 100 million bases of human data, much of which is redundant, and , unfortunately, with many errors in both original analysis and subsequent data transfer. Much effort in the first five years will be devoted to more efficient, more accurate and cheaper technologies for sequencing. There is also, even in the best hands, a 1% error. More complicating are the many minor normal variations (polymorphisms) with one human sequence differing on average every several hundred bases from another. Thus several genomes with obvious redundancy need be analyzed to sort out lab errors from these normal variations or from those causing disease. The big problem is differentiating a benign, insignificant polymorphism from a difference of a single base which could lead, for example, to sickling of red cells or to a viscous, pathological, bronchial mucus secretion (cystic fibrosis).</p>
		<p>The second five years would be devoted to closing the map with the final markers at a 1cM level with, hopefully, sequencing being done cheaply, efficiently and with minimum error and separating the frequent and widely scattered polymorphisms and other differences between individuals. Many of these will have much anthropological value in helping to delineate human evolutionary development. The last five years would be directed to sequencing the remainder of the genome and again, looking for the many inter-individual differences. The entire project is estimated to cost 3 billion 1991 dollars, minuscule in relation to agriculture or defense dollars, but yet a large sum of money. It is reasonable to guess that the entire bill may be ten-fold this amount. Francis Collins, director of the National Center for Human Genome Research and David Galas, former associate director of the office of Health and Environmental Research of the Department of Energy, revised and updated the plan (Collins and Galas, 1993). The results of the first five years have far surpassed all expectations, and Collins and Galas conclude that the future has even more to offer to scientific knowledge and the well-being of humanity.</p>
		<center>
			<h2><a name="996900"></a>Recent Happenings</h2>
		</center>
		<p>It is well beyond the scope of this chapter to try to summarize the developments in the genome project in 1991 and 1992. A dramatic example, though small in scope, is one offshoot, human gene therapy (see Chapter 8). There are, as of January 1993, about 3 dozen projects worldwide with one half having immediate life-saving results. Two school girls with a fatal immune disorder (adenosine deaminase deficiency) are back in class with the missing gene having been added to their white cells by researchers at the National Institutes of Health led by French Anderson and Michael Blaese. The location of this gene, its isolation and its characterization, are all results of the genome technology evolving from the genome project.</p>
		<p>Two human chromosomes, the male-determining Y and the second smallest autosome #21 have been almost totally mapped. The Y has 60 million bases, and David Page and colleagues at the Whitehead Institute in Boston have characterized and labeled 196 over-lapping human Y fragments grown in yeast. Likewise Daniel Cohen of Paris with a number of international collaborators have identified 191 markers along the 42 million bases of #21, the chromosome that bears Down Syndrome, as well as some potential Alzheimer's and other genes of much medical interest.</p>
		<p>Thus the yeast artificial chromosome (YAC) technology has been very catalytic to mapping progress, but another technical advance has provided even greater efficiency. As stated early in this chapter, the original variations detected by the RFLP technology were infrequent and not randomly scattered over the genome. The technology itself, as developed by Ray White, Helen Donis-Keller and others was slow, expensive, and not as informative as had been hoped. It had, however, shown that mapping was feasible and it brought the project a long way, allowing diseases such as Huntington, cystic fibrosis, Duchenne muscular dystrophy and many others to be mapped, permitting diagnosis and other medical applications. Humans have scattered through their entire genome, as do probably many other animals, clusters of repeated sets of 2, 3, or 4 bases called micro-satellites. Alec Jeffries in England had originally used similar clusters (variable number tandem repeats, &quot;VNTR's&quot;) as markers since they differ so much from location to location in the genome of a given individual and, more important, any single micro-satellite differs markedly from person to person unless they are related. Permitting this technology to be possible is another breakthrough, namely, the capacity to use nature's way of amplifying a minuscule sample of DNA, say from even a single cell, into enough DNA so it can be characterized by sequencing or by probes able to identify the sequence. This process is termed &quot;polymerase chain reaction&quot; or &quot;PCR&quot;. Thus sequence-tagged sites (&quot;STS's&quot;) can be located up and down the chromosomes with great specificity and efficiency and most of these would be the aforementioned micro-satellites. Confusingly, these are called by some &quot;simple sequence repeats or SSR's&quot;.</p>
		<p>By 1993, there were several major genome &quot;factories&quot;, Genethon in Paris directed by Daniel Cohen and 2 in the United States, one in Cambridge, Massachusetts led by Eric Lander and the other a consortium based at the University of Iowa. The aim of the Lander group, for example, was to characterize 10,000 STS's in the human genome and 4,000 to 6,000 in the mouse. The human map would thus have a density of one STS each 300,000 bases, or in genetic terms, about every 0.3 centimorgans. An immediate fall-out will be the use of these markers to dramatically decrease the potential error in genetic diagnosis as in amniocentesis.</p>
		<p>In October, 1992, a consortium of the National Institute of Health and CEPH in Paris published 1416 loci covering 95% of the human X chromosome and 92% of the remainder. So the science of the human genome project was well ahead of the proposed time table originally outlined by James Watson and the founding governmental and advisory committees.</p>
		<p>The National Institutes of Health, via the Center for Human Genome Research, initiated eleven &quot;Genome Centers&quot; between October 1990 and October 1992 in the United States. These included the two &quot;factories&quot; in Cambridge, Mass. and Iowa, mentioned above. Similarly, the Department of Energy created three centers of its own. </p>
		<p>Progress in the entire genome effort has far exceeded expectations and predictions. Some genetic diseases such as myotonic dystrophy have been found to be related to multiple repeats of simple sequences such as CGG and these repeated sequences are not only variable between individuals but may even be unstable from one generation to another. They have been termed &quot;premutational&quot; and in some diseases, such as Huntington's, this phenomenon explains why disease may become more severe from one generation to another. It should be added that the high variability of these repeats from one person to another has provided an excellent tool for linkage analysis as well as for forensic purposes.</p>
		<p>The Los Alamos-located gene base, GenBank, by early 1995, contained over 90,000 characterized bits of sequence from expressed genes in the human, about 30,000 from plants, and some 12,000 from invertebrates such as Drosophila. Since some genes are expressed in all tissues and others are limited to only specific tissues, Venter and his colleagues at TIGR have over 300 &quot;libraries&quot; of small portions of expressed genes from some three dozen human tissues, some embryonic and some cancerous.</p>
		<p>These small portions have been labeled &quot;EST&quot; for &quot;Expressed Sequence Tags&quot; since they are derived from sequencing the ends of genes expressed in various organs and tissues, In a &quot;Genome Directory&quot; published by Nature in 1995, Venter's group (TIGR) reports on some 55,000 EST's of which only about 10,000 are available in prior public data bases. They contain about 5 million sequenced base pairs, only 0.15% of the human genome (Maddox, 1995). The French group &quot;Genethon&quot;, lead by Daniel Cohen, has used the DNA collected by the Centre d'Etude Polymorphisme Humaine (CEPH) to prepare a physical map to position EST's along the human chromosome. To do this, they used the yeast artificial chromosome (YAC) technology which involves cloning large pierces of human DNA into yeast and then using overlapping stretches to piece together, like a large jig saw puzzle, the contiguous elements. Combination of the Venter data with those derived from Genethon is a major step forward in the total genome effort.</p>
		<p>A major contribution to the genome project has been the development of &quot;positional cloning&quot; (Collins, 1995). This technical breakthrough permits isolation and characterization of a given gene once its approximate location is known by standard genetic mapping techniques. It took over 8 years to isolate the Huntington's gene, but with positional cloning, by mid-1995, approximately 50 disease-related genes had been characterized. These include breast cancer, colon cancer, retinitis pigmentosa, several genes for diabetes and for Alzheimer's, and even a gene directly related to obesity in experimental animals (Zhang, 1994). </p>
		<p>Again, the &quot;genome&quot; movement is accelerating at a totally unexpected pace and the sequencing and characterization of the entire human genome will probably be achieved before the year 2000, if not sooner. The impact of this new knowledge on mankind is unpredictable.</p>
		<center>
			<h2><a name="996924"></a>Some Problems</h2>
		</center>
		<p>With the preceding history as background, it is easy to see that a major new science has arisen. Many think it is the most significant intellectual discovery in humanity's scientific evolution. But along with the new science are now a number of societal questions at all levels. In addition to the intellectual or scientific interests, the dominant and possibly the driving force in the development of the new genetic information are economic interests, but close behind in shaping this development are ethics and the law. However, the latter two serve more to check and balance the scientific and economic driving forces. Already major positions have been taken and the endeavor has already attracted sophisticated and concerned humanists as well as the usual spectrum of informed and uninformed whistle blowers (Kevles and Hood, 1993). On the economic side are the many new proprietary issues and even an entirely new industry made up mainly of small biotechnology companies supported by venture capital (Anderson, 1993). A number have already been swallowed up by larger firms and a few have already folded. </p>
		<p>Currently a major issue is patenting. Craig Venter, a scientist in the intramural NIH program, with the urging of the Director of the NIH, Bernardine Healy, applied for patents on 350 bits of DNA in June 1991 and another 2,735 pieces in February 1992. The first application raised a hue and cry amongst many scientists including James Watson with the feeling that patent protection would impede scientific progress. The British and other foreign scientists similarly objected; however, Sidney Brenner, a leading British molecular and developmental biologist, simultaneously applied for patents on a series of similar DNA fragments. In both, the pieces of DNA were beginning sequences of expressed genes (cDNA's) without knowledge of what was the gene or its role or where it is located. Is this simply a patenting of a piece of anatomy? Will it facilitate or impede progress in further mapping and sequencing of the human genome? In July 1992, Venter signed a contract with a venture-capital investment group to form a private, but not-for-profit, company to continue isolating and characterizing additional cDNA's, the Institute for Genome Research (TIGR). The Merck company has given Washington University in St. Louis a $10 million grant to do the same as the Venter group, but to provide the data to the public without the requirement of signing off on its commercial exploitation. By 1995, even religious leaders had been brought into the fray, led by Jeremy Rifkin, the renowned whistle-blower, and question the patenting of genes, gene sequences, animal components, and whole animals such as those added (&quot;transgenic&quot;) or deleted (&quot;knockout&quot;) genes (Stone, 1995).</p>
		<p>Politics again took precedent over administrative logic. James Watson was so bitterly opposed to the patenting of human gene sequences that he voiced his concerns at every opportunity, incurring the wrath of Bernardine Healy, his boss and Director of the National Institutes of Health. The result was Watson's requested resignation, bringing many in the scientific community to his defense. He was not replaced until April, 1993 when Francis Collins, MD, PhD was finally appointed. In essence it was another invasion into the control of science by the body politic. Currently, the issues of patenting and rights of ownership of the chemicals, of the animals and of the involved knowledge are being debated at all levels: scientific, legal, economic and, of course, from the ethical and social perspectives.</p>
		<p>Similarly, who will control an individual's genetic data as well as the hopefully anonymous pooled genome data which are critical for epidemiologic and anthropologic studies? Can these data be used in employment, marriage counseling, insurance applications, forensic identification, filing simply as a mechanism for general identification like a social security number, to name a few? Most important, who will decide the policy for the data's use? Should they be elected or appointed politicians, civil servants, lawyers, clergy, ethicists, physicians, economists, the public by referenda? In this volume we hope to provide a clear account of morality, so that no matter who decides, they will do so within morally acceptable limits. We also hope to clarify some conceptual issues, such as what counts as a genetic malady, so that together with the account of morality, morally acceptable public policies can be developed. We do not think that most of the moral problems that arise from the new genetic information will have unique answers, but we do think that there is a range of morally acceptable answers, and we hope to provide a way of determining what that range is. </p>
		<p>As world population expands and resources become rate-limiting, as is now happening, the economic issues will no doubt dominate, economics meaning the control of resources. Hopefully even these major problems will be based on a clear, comprehensive and commonly accepted moral foundation. Again, we do not claim to be able to provide the answers, only to provide a guide for determining which answers are morally acceptable and which are not. Clarifying the conceptual and moral issues involved is thus the purpose of this volume.</p>
		<p>Morality, man's best friend the dog, and some even greater problems ahead?</p>
		<p>One problem has already arisen with diabetes, not the common type afflicting the middle-aged and frequently managed by diet, weight loss and exercise alone, but the type that afflicts mainly children at the time of adolescence. The science is now at the point where children can be spotted ahead of time with some degree of statistical confidence; there are complicated and potentially dangerous therapies, and the entire problem is very complex emotionally, economically and sorely needing humanitarian guidance. As big a problem as it is, with about 1 in every 300 American children by age 18 taking insulin daily to stay alive, it may well be diminished by some of the future problems the genome project could introduce. Nevertheless, Type I diabetes is an excellent example of today's interplay between health policies, economics, genetics and the moral issues these raise, and in a young and productive population. However, let us focus on some impending problems on the horizon, and probably not as far off as most expect. </p>
		<p>In the first paragraph of this chapter on the history of the genome, the impact of selective breeding on both the body structure and the behavior of the dog was introduced. Some 4 or 5 decades ago, the geneticist C.C. Little and the then Director of the Rockefeller Foundation, Alan Gregg, discussed the heredity of behavioral characteristics. This led to a 13 year study by two scientists well versed in canine psychological testing as well as in genetics, John Paul Scott and John L. Fuller (Scott &amp; Fuller, 1965).</p>
		<p>Five strains of dogs were selected for their relatively small size as well as for their disparate behaviors. These were the cocker spaniel, the African basenji, the wire-haired fox terrier, the Shetland sheep dog and the standard beagle. Crosses and back-crosses were made and the offspring studied for a large number of dog behaviors. Many traits clearly and significantly segregated genetically. For example, the basenji is nippy and is handled on a leash with difficulty; the cocker the reverse. This jitteriness was easily separated in hybrids, and the trait appeared to be inherited by a single gene. Any dog breeder would acknowledge that even within a given breed, behavior is not only stereotypical for that breed but even variants within the breed are clearly inherited. Untrained Shetlands or border collie pups will &quot;herd&quot; a single child who has left a group of children back to the group and, if the child is reluctant, will crouch and stare the child into submission, just as it would do to a recalcitrant lamb. Likewise a Labrador pup offered a bowl of water will not drink; it will climb in!</p>
		<p>With the conservation of genes across evolutionary boundaries, one may ask what are the genes in human beings homologous to those controlling many of the behavioral traits in the dog. In what centers in the brain are these genes expressed, what is the nature of the proteins, what neurotransmitters or receptors are involved, what is the relative level of their expression, and many other questions? The dog has 39 pairs of chromosomes and some preliminary assignment of genes have been made to simple linkage groups. Two investigators, Jasper Rine at the Department of Energy Lawrence Laboratories in Berkeley, California and George Brewer at the University of Michigan have independently started using the most recent technology (polymerase chain reaction (PCR) and micro-satellites) for mapping the dog genome in preparation for a number of studies on dog diseases as well as on their behavioral traits.</p>
		<p>These dog studies will permit not only identity of genes in the dog but also, as above, the homologous genes in human beings. On the beneficial side the search for psychotropic pharmacologic agents will receive a tremendous boost, and it is a reasonable guess that the pharmaceutical industry will capitalize greatly on these studies. On the other hand, there are already enough problems dealing with the moral and ethical issues raised by the genome in the area of somatic-organic disease like diabetes, Huntington Disease and cancer. As one enters into the field of the hereditary aspects of human behavior and other brain functions such as intelligence, capabilities to perform unique tasks involving mathematics, music, spatial recognition or simply physical coordination, the ethical problems are compounded manifold. Consider, for example, the preliminary studies which suggest that inherited factors play a role in determining sexual orientation (Hamer et al, 1993). Gay/Lesbian rights activists are concerned that this may be construed that this genetic difference is something that should or could be reversed. Likewise, a significant pedigree has been published reporting a pattern of sex-linked inheritance of pathological aggression in males in the kindred (Anon, 1993).</p>
		<p>Should a moratorium be proscribed on all behavioral research and genetics? Some may feel strongly that there should be. On the other hand, when one deals with a patient in and out of severe depression and sees how impotent modern drug therapy may be in certain cases, one is compelled to encourage the science along in the hope that some intervention may specifically alter this pathologic behavior before suicide terminates the problem. The dog genomic studies and even those looking at some very simple behavior patterns in fruit flies will certainly facilitate this area of neuroscience. But moral guidelines are imperative; no rational person will question this obvious conclusion. Many of the following chapters present current problems and the moral issues they raise. The dog behavioral studies just mentioned are an example of the even greater and more complicated problems of the future. To use the vernacular, we are already in deep water with the ethical problems the genome provides relative to physical disease, but far deeper waters lie ahead when we consider intellectual and behavioral inheritance and the genes involved.</p>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h2><a name="996949"></a>References</h2>
		</center>
		<ol>
			<li>Anderson C. Genome Project Goes Commercial. Science 259:300-02, 1993. 
			<li>Anon. Evidence Found for a Possible &quot;Aggression Gene&quot; Science 260:1722-23, 1993. 
			<li>Botstein D, White RL, Skolnick M, Davis RW. &quot;Construction of a genetic linkage map in man using restriction fragment polymorphisms.&quot; Am J. Human Genetics 32: 314-31,1980. 
			<li>Collins F, Galas, D. &quot;A New Five-Year Plan for the U.S. Human Genome Project.&quot; Science 262: 43-49, 1993. 
			<li>Collins, F. &quot;Positional Cloning Moves from Perditional to Traditional.&quot; Nature Genetics 9: 347-350, 1995. 
			<li>Cook-Deegan RM: &quot;The Human Genome Project: Formation of Federal Policies in the United States, 1986-1990&quot;. in Biomedical Politics. Hanna, K. E., ed. National Academy Press. pp. 99-168. 1991. 
			<li>Fleischmann, RD et al. &quot;Whole Genome Random Sequencing and Assembly of Haemophilus influenzae Rd.&quot; Science 269: 496-512. 1995. 
			<li>Genome Directory. Nature. In press. 
			<li>Hamer DH, Hu S, Magnuson VL, Hu N, Pattatucci AML. &quot;A Linkage Between DNA Markers on the X Chromosome and Male Sexual Orientation.&quot; Science 261:321-27, 1993. 
			<li>Hunkapiller T, Kaiser RJ, Koop BF, Hood L. &quot;Large-scale and Automated DNA Sequence Determination.&quot; Science 254: 59-67, 1991. 
			<li>Kan YW, Doizy AM. &quot;Polymorphism of DNA Sequence Adjacent to Human Beta-Globin Structural Gene: Relationship to Sickle Mutation.&quot; Proc. Nat. Acad. Sci. 75: 5631-35, 1978. 
			<li>Kevles DJ. In the Name of Eugenics: Genetics and the Uses of Human Heredity. Univ. of California Press, Berkeley, CA 1985. 
			<li>Kevles DJ, Hood L. eds. The Code of Codes: Scientific and Social Issues in the Human Genome Project. Harvard University Press, Cambridge, 1993, 397 pp. 
			<li>Lee T. The Human Genome Project: Cracking the Genetic Code of Life. Plenum, New York, 1991, 360 pp. 
			<li>Lee TF: Gene Future: The Promise and Perils of the New Biology. Plenum Press, New York, 1993 in press. 
			<li>Lindee, MS. Suffering Made Real: American Science and the Survivors of Hiroshima. Chicago: University of Chicago Press, 1995. 
			<li>Maddox, J. &quot;Directory to the Human Genome.&quot; Nature. 376: 459-60. 1995 
			<li>McKusick VA. Mendelian Inheritance in Man. Baltimore: Johns Hopkins University Press. 1994. 
			<li>McKusick VA. &quot;Mapping and Sequencing the Human Genome.&quot; New Engl. J. Med. 230: 910-15, 1989a. 
			<li>McKusick VA. &quot;The Human Genome Organization: History, Purposes, and Membership.&quot; Genomics 5:385-87, 1989b. 
			<li>Muller-Hill B. &quot;The Shadow of Genetic Injustice.&quot; Nature 362: 491-92, 1993. 
			<li>Scott JP , Fuller JL. Dog Behavior. The Genetic Basis. University of Chicago Press, Chicago 1965, 468 pp. 
			<li>Stone, R. &quot;Religious Leaders Oppose Patenting Genes and Animals.&quot; Science 268:1126, 1995. 
			<li>Tauber AI, Sarkar S. &quot;The Human Genome Project: Has blind Reductionism Gone Too Far?&quot; Perspectives in Biol. &amp; Med. 35: 221-36, 1992. 
			<li>Watson JD, Crick FHC. &quot;Genetic Implications of the Structure of Deoxynucleic Acid.&quot; Nature 171: 737-38, 1953. 
			<li>Zhang, et al. &quot;Positional Cloning of the Mouse obese gene and its Human Homologue.&quot; Nature. 372: 425-32. 1994. 
		</ol>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h3><a name="997005"></a>General Medical Resources on Genetics</h3>
		</center>
		<ol>
			<li>Genbank, Los Alamos National Laboratory, DOE, Los Alamos NM (a computerized repository of DNA sequences derived from many organisms including humans). 
			<li>Human Genome Data Base. John Hopkins University, 1830 East Monument St. Baltimore, MD 21205 (a computerized data base). 
			<li>McKusick VA : Mendelian Inheritance in Man: Catalogs of autosomal dominant, autosomal recessive and X-linked phenotypes. 2 volumes. 11th ed. Johns Hopkins University Press, Baltimore, 1994, 3009 pp (a comprehensive listing all known hereditary human diseases and non-disease traits). 
			<li>O'Brien SJ ed: Genetic Maps: Locus Maps of Complex Genomes, 5th ed. Cold Spring Laboratory Press, Long Island, NY 1990 (a catalogue of chromosomes and gene locations in all organisms so far studied including humans and subhuman primates). 
			<li>Scriver CR, Beaudet AL, Sly WS, Valle D (eds): The Metabolic Basis of Inherited Disease, 3 Volumes. 7th ed. McGraw-Hill, New York, 1995, 5248 pp. (a detailed series of chapters on hereditary diseases in human beings for which the molecular and biochemical bases have been defined). 
			<li>Thompson MW, McInnes RR, Willard HF: Genetics in Medicine, WB Saunders &amp; Co, Philadelphia, 1991, 500 pp (a general text for human genetics). 
		</ol>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h3><a name="997021"></a>Significant Organizational Milestones in the Evolution of the Genome Project</h3>
		</center>
		<ol>
			<li>Office of Technology Assessment. Mapping our genes- the Human Genome Projects: How big, how fast? OTA-BA-373. US Govt. Printing Office, April, 1988. 
			<li>National Academy of Science: Report of the Committee on Mapping and Sequencing the Human Genome. National Academy Press, Wash. DC., 1988, 116 pp. 
			<li>Department of Energy: Health and Environmental Research Advisory Committee (HERAC). The Human Genome Initiative, Dept. of Energy, April 1987. 
			<li>Department of Energy; The Human Genome, DOE/ER-0544P, 1992. 
			<li>National Academy of Science: DNA technology in forensic science, Nat. Acad. Press, Wash, DC, 1993, 200 pp. 
		</ol>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h3><a name="997034"></a>Periodicals</h3>
		</center>
		<ol>
			<li>Human Genome News, published by NIH and DOE. HGMIS, Box 2008, Oak Ridge National Laboratory, Oak Ridge, TN 37831-6050. 
			<li>Progress: Human Genome Project, published by the National Center for Human Genome Research, NIH, Bethesda, MD. 
			<li>Genomics, Academic Press, Orlando, Florida, monthly. 
			<li>Nature: Genetics monthly. 
		</ol>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h3><a name="997045"></a>Other Books on the Genome</h3>
		</center>
		<ol>
			<li>Bishop JE, Waldholz M. Genome Simon &amp; Shuster, New York, 1990, 352 pp. 
			<li>Jones S. The Language of the Genes: Biology, History and the Evolutionary Future, Doubleday, New York, 1993. 
			<li>Wills C. Exons, Introns and Talking Genes: The Science Behind the Human Genome Project, Basic Books, Harper Collins, New York 1991, 368 pp. 
			<li>Wingerson L. Mapping Our Genes, Dutton, New York, 1990. 
		</ol>
		<p>
		<hr width="50%">
		</p>
		<center>
			<h3>Biology 4: Genes and Society (<a href="index-b4-97.html">1997</a>, <a href="../bio4-1998/index-b4-98.html">1998</a>)</h3>
		</center>
	</body>

</html>
