<base href="http://www.accessexcellence.org/LC/SS/PS/PCR/PCR_technology.html">
<html> <head>
<title>PCR Technology</title><noindex><!-- Start Quantcast tag -->
<script type="text/javascript" src="http://edge.quantserve.com/quant.js"></script>
<script type="text/javascript">
_qacct="p-00jJ5sUyRZnac";quantserve();</script>
<noscript>
<img src="http://pixel.quantserve.com/pixel/p-00jJ5sUyRZnac.gif" style="display: none" height="1" width="1" alt="Quantcast"/></noscript>
<!-- End Quantcast tag -->

<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3096596-1";
urchinTracker();
</script>



<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-3096596-1";
urchinTracker();
</script>

</noindex>





<link rel="stylesheet" type="text/css" href="/styles.css">
</head>

<body>
<h1>PCR Technology</h1>

<em><strong>Connie Veilleux</strong></em>
<p>
<hr>
<p>


<h2>Introduction</h2>

Polymerase chain reaction (PCR) has rapidly become one of the most
widely used techniques in molecular biology and for good reason: it is
a rapid, inexpensive and simple means of producing relatively large
numbers of copies of DNA molecules from minute quantities of source
DNA material--even when the source DNA is of relatively poor quality.
<p>

PCR involves preparation of the sample, the master mix and the
primers, followed by detection and analysis of the reaction products.
These steps are discussed below.
<p>

<h2>Sample Preparation</h2>

PCR is very versatile.  Many types of samples can be analyzed for
nucleic acids.  Most PCR uses DNA as a target, rather than RNA,
because of the stability of the DNA molecule and the ease with which
DNA can be isolated.  By following a few basic rules, problems can be
avoided in the preparation of DNA for the PCR.  The essential criteria
for any DNA sample are that it contain at least one intact DNA strand
encompassing the region to be amplified and that any impurities are
sufficiently diluted so as not to inhibit the polymerization step of
the PCR reaction.
<p>

Although any protocol is acceptable for PCR purposes, it is often best
to use the fewest steps possible in DNA preparation in order to
prevent accidental contamination with unwanted DNA.  Usually a 1:5
dilution of the sample with water is sufficient to dilute out any
impurities which may result from the purifying protocol.
<p>

The simplest method of isolating DNA from cells is as follows:

<ol>
  <li>Cells can be obtained by using a toothpick to scrape under the
fingernails, swabbing the inside of the mouth or from the roots of
plucked hairs.  Regardless of source, cells are resuspended in 20 ul
of water.  Skip to step four.
<p>

  <li>If you are using cells suspended in media, centrifuge at 1200- 1500Xg
for 5 minutes.  Resuspend the cell pellet in 1 ml of phosphate
buffered saline (PBS) and repellet by spinning at 1200- 1500Xg for 5
minutes.  Repeat.  These PBS washes remove medium, and its inhibitory
factors, from the surface of the cells.  After the last wash resuspend
the cell pellet in 20 ul of distilled water. Be aware that too much
cell debris can inhibit the PCR reaction.  If this happens, it may be
necessary to further dilute the DNA sample.  Go to step four.
<p>

  <li>For bacterial samples take a toothpick and scrape the teeth, or
swab the throat, ears or between the toes.  Resuspend material in
500ul of water.  Freeze and thaw sample three times with vigorous
shaking or vortexing between repetitions to break the bacterial cell
wall.  Although not all DNA will be released from the cells, there
will be a sufficient quantity for PCR.  Go to step four.
<p>

  <li>Place the sample in a 95oC heating block, or in boiling water, for 5
minutes.  This step inactivates the DNase molecules that are found in
the sample preparation.  If left intact, DNase could clip the desired
DNA template molecule into fragments which would be unsuitable for
PCR.  If there is very little DNA in the sample preparation, the DNA
can be concentrated by ethanol precipitation. The sample is now ready
for PCR.

</ol>


DNA samples for PCR--regardless of preparation method--are generally
run in duplicate in order to provide a control for the relative
quality and purity of the original sample.  Adding a small amount of
DNA to the control just after the master mix step allows the detection
of anything in the completed sample prep which would inhibit the PCR
reaction.
<p>


<h2>Preparation of Master Mix</h2>

The Master Mix contains all of the components necessary to make new
strands of DNA in the PCR process.  The Master Mix reagents include:
<p>

<pre>
Final      Component	Purpose 
Conc.
           Water

1X         Buffer       keeps the master mix at the proper pH so the PCR
                        reaction will take place. 

200uM      Deoxynu-	provide both the energy and nucleosides for the 
           cleotides    synthesis of DNA.  It is important to add equal
                        amounts of each nucleotide (dATP, dTTP, dCTP,
                        dGTP) to the master mix to prevent mismatches of
                        bases. 

0.2-1.0uM  Primers	Short pieces of DNA (20-30 bases) that bind to the
                        DNA template allowing Taq DNA polymerase enzyme to
                        initiate incorporation of the deoxynucleotides.
                        Both specific and universal primers can be used.

2.5U/100ul AmpliTaq 	A heat stable enzyme that adds the
           polymerase	deoxynucleotides to the DNA template.
	

0.05-1.0ug Template 	The DNA which will be amplified by the PCR
           DNA		reaction.

</pre>

<h2>Notes on the Master Mix</h2>

The Master mix buffer is often stored as a
10X stock solution (100 mM Tris-HCL, pH 8.3, 500 mM KCL, 1.5 mM MgCl2)
which is diluted to 1X for use.  Both the Master mix buffer and the
purified water can be stored at room temperature.  Store
deoxynucleotides, primers and Taq DNA polymerase enzyme at -20oC.
<p>

Although 100ul of master mix per reaction is generally used, it is
possible to use as little as 25 or 50ul to save on cost of reagents.
Regardless of the total volume, be certain to keep the final
concentrations of reagents constant.
<p>

Master mix reagents can be optained from a variety of companies.
Often the initial concentration of the reagent will differ depending
on which company produced it.  It is easy to figure out how much stock
reagent to use by following a simple formula:
<p>

<pre>
(initial concentration) X ( volume needed ) =
                                    (final concentration) X (volume of sample)
</pre>

For example: I have 10X buffer, 10 mM of each nucleotide, 0.5 mM
primers and Taq DNA polymerase at 5 Units/ul.  I want to make one 50
ul reaction.  Calculations are as follows:
<p>

10 X buffer:            (10X) X (5 ul) = (1X) X (50 ul)

Nucleotides:            (10,000 uM) X (1 ul) = (200 uM) X (50 ul)
(10mM=10,000uM)
    
primers                 (500uM) X (O.1ul)= (1.0uM) X (50 ul)
                        Since it is impossible to pipet 0.1ul
                        accurately, a dilution needs to be made first.
                        Add 10 ul of stock primer solution to 990 ul of
                        water to get 5uM concentration of primers. This
                        new primer dilution can be stored at 4oC.
                        Calculation for 5uM stock:
                        (5uM) X (10 ul) = (1.0 uM ) X (50 ul)

Taq  DNA polymerase	(5Units/ul) X ( 0.25 ul) = (.025 Units/ul) X (50 ul)
 2.5 Units/100ul=	Since it is impossible to pipet 0.25ul accurately, a
 .025 Units/ul		dilution needs to be made first.  Add 1.25 ul stock
                        to 3.75 ul water to get a 1.25 Units/ul concentration.
                        Discard and make fresh with each use.
                        Calculation for 1.25 Units/ul stock:
			(1.25 Units/ul) X (1 ul) = (.025 Units/ul) X (50 ul)
</pre>
    
To make the master mix for one reaction add: 
<ul>
  <li>5 ul  10X buffer
  <li>4ul Each nucleotide (1ul each of dATP, dCTP, dGTP, dTTP))
  <li>20 ul  Each primer (10ul of each)
  <li>1 ul Taq  DNA polymerase (Total volume = 30ul) 
  <li>add 15 ul of water 
  <li>5 ul of template  (Total volume = 50 ul)
</ul>

If want to make 3 reactions, 3 X 50ul = 150ul. Use this number in the
formula for "volume of sample."
<p>

<h2>Primers</h2>

A primer is a short segment of nucleotides which is complementary to a
section of the DNA which is to be amplified in the PCR reaction.
Primers are annealed to the denatured DNA template to provide an
initiation site for the elongation of the new DNA molecule.  Primers
can either be specific to a particular DNA nucleotide sequence or they
can be "universal." Universal primers are complementary to
nucleotide sequences which are very common in a particular set of DNA
molecules.  Thus, they are able to bind to a wide variety of DNA
templates.
<p>

Bacterial ribosomal DNA genes contain nucleotide sequences that are
common to all bacteria.  Thus, bacterial universal primers can be made
by creating primers which are complementary to these sequences.<br>
Examples of bacteria universal primer sequences are:<br>
Forward  5' GAT CCT GGC TCA GGA TGA AC 3'  (20 mer)<br>
Reverse 5' GGA CTA CCA GGG TAT CTA ATC 3'  (21 mer)
<p>

Animal cell lines contain a particular sequence known as the "alu
gene".  There are approximately 900,000 copies of the alu gene
distributed throughout the human genome, and multiple copies
distributed through the genome of other animal cells, as well.  Thus,
the alu gene provides the sequence for a universal primer for animal
cell lines.  The alu primer is especially useful in that it binds in
both forward and reverse directions.<br>
The alu universal primer seqeunce is as follows:<br>
5' GTG GAT CAC CTG AGG TCA GGA GTT TC 3' (26mer)
<p>

When using universal primers the annealing temperature on the thermal
cycler is lowered to 40-55 degrees C.
<p>

Sometimes primer units are listed in optical density reading (OD).  If
this is a problem you will need to convert to molarity using the
following equations: Change optical density reading of primer to
molarity (uM units)-

<ol>
  <li> N = # of primer bases
  <li> SIGMA 260 =~ 10,000 X N/ m X cm
  <li> Molecular weight =~ 330 X N
  <li> OD<sub>260</sub> / SIGMA 260 X 10<sup>6</sup> = Concentration (uM)
</ol>

For example- primer is 20 bases long/ OD<sub>260</sub> = 10.

<ol>
  <li> N = 20
  <li> SIGMA 260 =~ 10,000 X 20/m X cm = 20,000/m X cm
  <li> molecular weight =~ 330 X 20 = 6,600
  <li> 10 OD<sub>260</sub>/20,000 m<sup>-1</sup>cm<sup>-1</sup> X 10<sup>6</sup> = 50uM
</ol>

<h2>Detection and analysis of the reaction product</h2>

The PCR product should be a fragment or fragments of DNA of defined
length. The simplest way to check for the presence of these fragments
is to load a sample taken from the reaction product, along with
appropriate molecular-weight markers, onto an agarose gel which
contains 0.8-4.0% ethidium bromide.  DNA bands on the gel can then be
visualized under ultraviolet trans-illumination.  By comparing product
bands with bands from the known molecular-weight markers, you should
be able to identify any product fragments which are of the appropriate
molecular weight.

<!-- begin-ae-footer -->
<hr>

<center><h5><a href=../../../../search.php>Search</a> | <a href=../../../../WN/>What's News</a> | <a href=../../../../AB/>About&nbsp;Biotech</a> | <a href=../../../../message.php>Teacher's&nbsp;Lounge</a> | <a href=../../../../RC/>Resource&nbsp;Center</a> | <a href=../../../../AE/>Activities&nbsp;Exchange</a> | <a href=/LC/>Teacher/Scientist&nbsp;Network</a></h5></center>
<center><h5><a href=../../../../index.php>Access Excellence</a></h5></center>
<hr>
<!-- end-ae-footer -->

<!--plugger-end-shell-footer-LC-->


<!--php-begin-ae-nhm-survey-->

<!-- BEGIN SURVEY POPUP CODE -->

<!-- END SURVEY POPUP CODE -->

<!--php-end-ae-nhm-survey-->

</body> </html>




